---
layout: page
title: News and Press
menu: main
---

## The Science special issue with two papers from the group

<p><span class="image right"><img src="images/science_primateissue.png" width="70"  alt="" /></span>Detailed acount of baboon interspecies relationship with Ph.D. student Erik Fogh as shared first author.
</p>

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; June. 2. 2023</i></p>


## John Hawks writes about our recent work on human X chromosomes

> **Could meiotic drive explain why Neandertal ancestry is so sparse on the X chromosome?** <br> New research provides a surprising insight into the out of Africa dispersal, which may change the way we think about Neandertals and Denisovans.

Read the full article [here](https://johnhawks.net/weblog/positive-selection-x-chromosomes-recent-human-evolution/)

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; March. 11. 2023</i></p>


## An X file - the truth is out there

<p><span class="image right"><img src="images/xfiles_cover.png" alt="" width="100"  /></span>
Tracing my ancestry into the past spreads my genes thinly across ancestors as these double generation by generation. Each gene thus travels a separate line of ancestors that serve as vessels carrying it through the ages. The only genes I <it>can</it> have are ones that successfully made it all the way from one generation to the next. That is all there is to natural selection, really. It works on genes, not organisms. Organisms are just vessels, and their fitness only comes into the picture because a gene's contribution to better organism fitness helps the gene get passed to more new vessels in the next generation. If a gene could somehow pass to more new vessels by <it>reducing</it> organism fitness, it would not break the logic of natural selection. Strange as it may seem, this actually happens.
</p>

I have two copies of each chromosome except the X chromosome, which only women carry in pairs. My X chromosome instead pairs up with a small Y chromosome. Unlike other pairs of chromosomes, the X and Y chromosomes do not mix their genes before each one ends up in a sperm cell. So if a gene on the X chromosome can somehow reduce the viability or motility of the sperm cells with Y chromosomes, it will pass to more offspring in the next generation. Killing off sperm cells reduces fertility and skews the sex ratio of children, reducing the individual's fitness. Yet, the advantage enjoyed by the selfish gene variant may be so big that it more than makes up for the lower fitness of the organism that carries it. The phenomenon is called meiotic drive, but it is natural selection nonetheless. Researchers of model organisms have found this particular kind of selection in mice and fruit flies but, so far, not in humans. 

In a project we publish today in [Cell Genomics](https://doi.org/10.1016/j.xgen.2023.100274), we report genetic evidence of strong selection along the human X chromosome. when a selected gene spreads through a population, dragging its genomic neighborhood along, it "sweeps" away genetic diversity around it. Using comparative genome analysis, we searched for such "selective sweeps" on the X chromosomes of human males worldwide. We found 14 sweeps shared by all non-African populations, suggesting that this selection happened after the main migration out of Africa but before the subsequent spread of people across Eurasia (using ancient DNA, we narrowed this interval to 55,000-45,000 BP). Surprisingly the swept regions are without the small segments of Neanderthal DNA otherwise spread across non-African genomes. This absence suggests that the selected gene variants were all introduced from a contemporary human population not exposed to Neanderthal admixture. Although the mechanism of this strong selection remains unknown, it must be unique to the X chromosome. The working hypothesis guiding our continued effort is that it is indeed a meiotic drive. 

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Feb. 28. 2023</i></p>


## Research covered by New Scientist 
New Scientist [writes about the strong selection](https://www.newscientist.com/article/2348335-modern-humans-evolved-a-selfish-x-chromosome-after-africa-exodus/) we find on the X chromosome of non-Africans. The title is pushling it though.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Nov. 23. 2022</i></p>

## What are you waiting for?
<p><span class="image right"><img src="images/stream.jpg" alt="" /></span>Imagine standing on the bank of a stream, trying to spot a sequence of stepping stones that will take you to the other side without getting your feet wet. In your imagination, you draw lines between stepping stones close enough to jump between. The connected stones create a network of many alternative paths connecting the two banks. Standing on one stone with more than one other stone to jump to, you probably prefer some over others, and you will tend to linger for longer if the next jump seems demanding. Having assessed the difficulty of jumps in a network of connected stones, you will know roughly how long the crossing is expected to take. What you just made in your head is called a phase-type distribution.
Phase-type distributions describe the time it takes to proceed through a series of connected states until whatever state constitutes the end. This applies to a vast range of situations, as you can probably imagine. For example, phase-type distributions describe everything from stream crossings to the patterns relatedness among DNA sequences. The properties of phase-type distributions are well described using matrix algebra. However, their practical use has been limited because matrix computations are too slow to cope with many transient states. In other words, if the number of states is large, it takes so long to compute the expected time that no one cares to try. 
In the paper published today in <a href="https://link.springer.com/article/10.1007/s11222-022-10174-3">Statistics and Computing</a>, we present a set of algorithms that treat phase-type distributions as a network or graph just like the stepping stone example. This lets us compute many phase-type distributions orders of magnitude faster than before, greatly expanding their use. Whatever it is you are waiting for, chances are that the wait is phase-type distributed. Now it can be computed while you wait.</p>

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Nov 1. 2022</i></p>


## SAP lives forever
The assignment of DNA from organic material to species or taxonomic groups is integral to a number of scientific disciplines. Metagenomics is an approach particularly suitable for viruses and bacterial species, which have a relatively small genome. The approach can be used to characterize environments according to their genetic fingerprint. Even without taking genomic approaches, however, DNA sequencing of selected markers from environmental samples may provide ecological information or identify relevant species such as human pathogens. A related field where unknown specimens are identified based on Cytochrome Oxidase I (COI) has become known as DNA Barcoding. SAP is a tool answer the central question in these fields: what taxonomic group does an unknown organism represented by a DNA sequence belong to? SAP uses a Bayesian approach to calculate a probability distribution over all taxa represented in a sequence database. The probability of assignment to each taxa serves as a measure of confidence in the assignment. SAP is available as both a [command line tool](https://services.birc.au.dk/sap/downloads) and a [web service](http://services.birc.au.dk/sap). Thanks to clever containerization technology that was not available when SAP was published, the SAP program is now available as a [docker image](https://hub.docker.com/repository/docker/kaspermunch/sap) for hassle-free installation on all platforms.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Oct 3. 2021</i></p>

<!-- 
## New Group member
Welcome to PhD student Erik Fogh Sørensen. 

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Aug 1. 2020</i></p>
 -->

## Evolution and domestication of the lama and alpaca
The paper on domestication and post-Spanish conquest evolution of the llama and alpaca is out in Genome Biology.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; July 2. 2020</i></p>

## Spotlight 
David Castallano's paper on the distribution of fitness effects in the great apes is chosen by Genetics for their yearly spotlight publication.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Feb 7. 2020</i></p>

<!-- 
## Book contribution from the group
"Population genomics in the great apes" out in [Statistical Population Genomics](https://www.springer.com/gp/book/9781071601983) from Springer.  

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Jan. 27. 2020</i></p>
 -->

## Funding from the Novo Nordisk Foundation
The group was awarded 2.5 mill DDK from the Novo Nordisk Foundation. The sequencing of full genomes from thousands of humans allow us to revisit the gene-centric view of evolution popularized by Richard Dawkins in his 1976 book “The Selfish Gene”. The funded research will address the fundamental question: how do the selfishness of human genes influence genome evolution, cellular function, and human health? Selfish genes in other organisms are found on sex-chromosomes. They act by distorting male meiosis and spermatogenesis. This distortion results in an uneven proportion of sperm that carry each sex chromosome and may result in partial or full infertility. Selfish genes may thus be promoted by natural selection even if harmful to the organism. We believe this that selfish genes can account for unexplained patterns of human genetic diversity on the X chromosome. W will will use computer simulations and analysis of full genomes from across the world to identify candidate selfish genes involved in sex chromosome evolution and male fertility. 

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Nov 29. 2019</i></p>

## Distribution of fitness effects in the great apes
David Castallano's paper on the distribution of fitness effects in the great apes is out in Genetics.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Nov 1. 2019</i></p>


## Mutation rate and selection in the homininae
David Castallano's paper on how mutation rate and selection at linked shapes diversity in humans and other homininae is out in Genome Biology and Evolution.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Oct 19. 2019</i></p>

<!-- 
## Mix yourself a baboon

Just as when a new cocktail is invented by mixing old ingredients, evolution sometimes makes a brand new species by mixing old ones. One such remarkable example is the Kinda baboon that inhabits large parts of Angola, Zimbabwe, and Dem. rep. Congo. Two ancestral baboon species had followed separate evolutionary paths for about one and a half million years until they met only 150 thousand years ago and fused to create the Kinda baboon. This surprising finding is part of a study that we just published in Science Advances, as part of a large international collaboration with important contributions from the Mailund and Schierup labs at BiRC.

By analyzing full genomes from multiple individuals from each baboon species, the study reveals the complicated ancestral relationship between the six baboon species. The species tree of baboons has two main clades, one with northern species (Guinea, Olive, and Hamadryas) and one with the southern species (Yellow, Kinda, and Chacma). The species tree has bifurcations where ancestral species split into two, just like we usually think of a species tree, but it also has interconnections between branches, where branches fuse rather than split. One such case is the fifty-fifty mix of a northern and a southern ancestral species giving rise to the Kinda baboon.

As primates, baboons are close relatives to humans and studying them is a way to understand the evolutionary forces that shape our own species. Baboon species are about as different from each other as modern humans were from Neanderthals and other contemporary archaic humans, and just as Neanderthals hybridized with modern humans, so do living baboon species. Although very different in terms of looks, baboon species interbreed where their ranges overlap and the published study reveals ongoing admixture between Yellow and Olive Baboons. Further studies of admixture between living baboon species may thus reveal important insights, that will help us understand our own complicated history of admixture with extinct human species such as the Neanderthal. -->

 
## Elephants and the mesh of life
<p><span class="image right"><img src="images/straight_tusked.png" alt="" /></span>The evolution and diversification of life are nicely depicted by a tree – with the original first living organism at the root and the main branches splitting into still smaller branches and twigs that eventually lead to the living species at the leaves. But what about all the species that went extinct, you may ask. The dinosaurs found themselves in dire straits 65 million years ago. In fact, species go extinct all the time. Whereas recently emerged species most likely persist, species that arose hundreds of million years ago have most likely gone extinct since then. This continuously removes twigs and branches from the tree of life, which is why it has ended up looking like, well, a tree.</p>

Elephants are an excellent example of a group of species where some have gone extinct and others not. There are three living species: the Asian elephant and the forest and savanna elephants in Africa. However, we would only need to travel back 100.000 years to meet another three species of elephants: The straight-tusked elephant, the woolly mammoth, and the Colombian mammoth. Fortunately, it has been possible to obtain enough DNA from bones of these extinct species, that their genomes could be sequenced. Over the past year or so, I have been part of an effort led by the excellent Elle Palkopoulou, to analyze the full genomes of individuals from these six species.

In the study that we publish in PNAS today, we show that the species tree that relates these six species is crisscrossed by connections between the branches, making it look more like a mesh than a tree. When one species split into two, they diverge only slowly, and it seems to take at least a couple of million years to accumulate enough genetic differences that hybrids are no longer fertile. Until then, hybridization may happen, transferring genes from one species into the other. The most important finding in the paper is that elephant species hybridized extensively even a long time after they initially split apart. The most prominent example is the straight-tusked elephant (the model in picture). Most of its genes can be traced back to the initial split from the ancestor to African elephants, but a large proportion is derived from hybridization with forest elephants and with Mammoths. I bet that as more species groups are studied using full genomes we will see many more examples where hybridizations and even fusions of species make the species tree look more like a mesh than a tree.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212;  Feb. 26 2018</i></p>


## Waiting for good mutations
<p><span class="image right"><img src="images/mutation.jpg" alt="" /></span>We know intuitively that individuals from small isolated populations are often more related with each than individuals in large populations are. Whereas closely related individuals have a shared relative in the recent past, distantly related individuals will have to look back many many generations to find such a common ancestor. This leaves more time for genomic mutations to produce differences between the two individuals, and this is why individuals from large populations show more differences between their genomes than individuals from a small one do.</p>

So large population size go hand in hand with large genetic diversity. Species are isolated populations because they do not interbreed with the populations of other species. Among the great apes, bonobos and chimpanzees exhibit the least genetic diversity. Gorillas have larger populations and higher diversity only superseded by orangutans.

In each species, the genetic diversity is not even across the genome. Some regions of the genome have lower diversity because natural selection removes harmful mutations in regions, such as genes, of particular importance to the organism. This produces a landscape of higher and lower diversity that is shared between the great apes because the genes and other important regions are located in roughly the same places.

Temporary differences in this landscape of diversity may arise from events of adaptation. Once in while a beneficial mutation comes along, which gives the individuals that carry it an advantage in the natural selection. When such a mutation quickly spreads to the entire species the mutation and the region around it, becomes shared by all individuals – which obviously depletes genetic diversity in that part of the genome. Such depletions in diversity are called “selective sweeps” and supply evidence of how often new beneficial mutations are introduced. In a paper that is published in PNAS today, we use this principle to show that larger populations seem to experience such sweeps more often than small ones: orangutangs more so than gorillas more so than chimpanzees and bonobos. We think this reflects that adaptation is limited by how often good mutations arise in the population. If twice as many good mutations happen in a population with twice as many individuals, then twice as many sweeps should occur.

It turns out that humans are the great apes with the slowest rate of sweeps. You may think this is odd considering that we are six billion people. The reason is this only happened in the last ten thousand years or so, a mere blink of an eye to evolution. In the rest of our history, we were only a small crowd. – But maybe, from now on, we will not have to wait so long for the good mutations.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Jan. 30. 2017</i></p>

## Selective sweeps across twenty million years of primate evolution
Our paper in MBE shows how patterns of incomplete lineage sorting measures linked selection on evolutionary timescales. We show that a large portion of linked selection is due to selective sweeps and that the human-chimpanzee ancestor experienced a substantially higher frequency of sweeps than did the human-orangutang ancestor. These ancestral sweeps are enriched for sweeps in modern humans suggesting that several regions of the genome are repeatedly hit by sweeps.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Sept. 22. 2016</i></p>


## Great ape genetic diversity mapped
<p><span class="image right"><img src="images/grafic-bickerstaff-i-clee-ps-drass-j-gadsby-e-idoaga-a-sudmant-p.jpg" alt="" /> </span>The genomes of the great apes have been sequenced with the Gorilla and Bonobo genomes completing the picture last year. This gave us valuable new information about the patterns of genetic differences between humans and the other great apes and a detailed picture of the ancestral relationship of this group of species.</p>

Until now, however, we have lacked a comparable account of the differences of individuals within each species. As part of an international collaboration we publish today in Nature a deep survey of the genetic diversity of the species and subspecies of great apes. This shows that several of the apes including the Orangutan have genetic diversities that dwarf that of humans: where two humans show one difference for each thousand positions in the genome the orangutan sports two.The larger genetic diversity of apes allows us to peak further into the evolutionary past of these species to better understand the mechanisms shaping genomes as species evolve. The close relationship of humans and the other great apes means that a better knowledge of these processes will also over time contribute a better understanding of human evolution. Among other results analysis of the 78 genomes sequenced in this study has produced a first full picture of great ape evolution integrating within-species history with the more ancient history of how these species arose from common ancestors.

Knowledge of the patterns of diversity is also crucial in distinguishing the different subspecies of Chimpanzees, Gorillas and Orangutans and will aid zoos in planning of breeding programs to keep separate the different subspecies in captivity and thereby conserving their individual characteristics.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; July 25. 2013</i></p>


## Bonobo genome shows that bonobos and chimpanzees resemble us in separate ways
<p><span class="image right"><img src="images/ulindi.png" alt="" /></span>Most people think that the chimpanzee alone is our closest relative, but in fact there is a second ape, the bonobo, that we are just as related to. Humans split from the other apes 5-6 million years ago and only 1-2 million years ago the ancestor to bonobos and chimpanzees split into these two species. This likely happened when the Congo river formed, separating the two species – chimpanzees and bonobos do not swim. bonobos live south of the river in the Democratic Republic of Congo.</p>

Although darker and more slender than chimpanzees they are similar in appearance, but they Ulindidiffer strikingly from chimpanzees in social and sexual behavior. In chimpanzees the male is the dominant sex, hierarchy and conflict resolution is based on aggression. In bonobos females are dominant, hierarchy is based on mother’s status and conflicts are resolved through sex. These strikingly different behavioral characteristics have made scientists dub the bonobo “the hippie ape” and “the make love not war ape”. Our collaborators at the Max Planck Institute for Evolutionary Anthropology in Leipzig tell us that if they throw a toy into a chimpanzee cage the chimpanzees with fight over it leaving the winner to play with it alone. If it is thrown into a bonobo cage everyone will have sex and then share the toy. Several of the social and sexual traits that are only found in one of these apes are also found in humans. The frequent non-conceptive sexual behavior is thus shared with humans but not with chimpanzees, lethal aggression is found in chimpanzees and humans but not in bonobos humans and bonobos are very playful whereas chimpanzees are less so, and chimpanzees and humans apply cooperative hunting whereas bonobos do not.

Today the bonobo genome is published in Nature revealing a detailed account of our relation to bonobos and chimpanzees. The genomic DNA was obtained from the female bonobo Ulindi from Leipzig zoo while she was anesthetized anyway by the dentist. The Bioinformatics Research Centre has participated in the bonobo genome consortium and I have led an effort contributing a bioinformatic analysis of the bonobo genome, the results of which define the topic of the paper published today. The bioinformatical approach we apply is unique because it combines the analysis of genetics within species with the genetics between species, combining these two traditionally separated fields into one model.

Comparing the bonobo genome to the already known genomes of humans and chimpanzees we have shown that for some parts of the genome (about 3%) it is not bonobos and chimpanzees that are most similar but humans and bonobos or humans and chimpanzees. This patchwork of genetic relatedness results because the three species are so closely related that they may share genes inherited from the common ancestor of all three species. 25% of our protein coding genes has parts that are more similar to bonobo or chimpanzee than these are to each other. Future research may reveal if any of these genes are responsible for traits we only share with one of these apes. A candidate could be the TAAR8 gene that has a variant only shared between humans and bonobos. The gene codes for a protein receptor that is known to help mice smell amines in urine and may provide social cues in bonobos.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; June 13. 2012</i></p>

## Gorilla genome reveals our ancestry as genetic patchwork
<p><span class="image right"><img src="images/kamilah.jpg" alt="" /></span>The chimpanzee is our closest cousin in the tree of life. Humans and chimpanzees split into separate species about 5-6 million years ago. The gorilla is more remotely related to both humans and chimpanzees separating about 8 million years ago. Today the Gorilla genome is published in Nature revealing surprising properties of our own genetic history. As expected most of our genes are most similar to the ones of the chimpanzee, but it turns out that about many of our genes are more like gorilla genes than chimpanzee genes. The genetic material on chromosomes is mixed in sexual reproduction making each gene free follow its own path through evolution. Because human, chimpanzee and chimpanzee are quite equally related this results in mix of genes different kinds of relatedness. For most genes humans-and chimpanzees are most related, but 15% of chimpanzee genes are more related to the gorilla than to our genes and 15% of our genes are gorillas are more related to gorillas than to chimpanzees.</p>

We have elucidated this genetic patchwork that reflects the evolutionary history of humans, chimpanzees and gorillas. Analyzing these patterns has allowed us to firmly establish how important natural selection is to evolution of these species. They find that the genetic variation is smaller genes compared to the genome average because that natural selection removes mutations that disrupt gene function. The bioinformatical approach we apply is unique because it combines the analysis of genetics within species with the genetics between species, combining these two traditionally separated fields into one model. Our analyses also illuminate the process by which western and eastern gorillas split into separate sub-species. Although we commonly think of such speciations as happing at a distinct point in time, this does not always reflect reality. This study shows that the separation into western and eastern gorillas happened over an extended period of time.

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; March 7. 2012</i></p>


## No magic bullet explains Ice Age megafauna extinctions
<p><span class="image right"><img src="images/mammoth.png" alt="" /></span>Was it humans or climate change that caused the extinctions of the iconic Ice Age mammals (megafauna) such as the woolly rhinoceros and woolly mammoth? For decades, scientists have been debating the reasons behind these enigmatic Ice Age mass extinctions, which caused the loss of a third of the large mammal species in Eurasia and two thirds of the species in North America. In an extensive, inter-disciplinary research collaboration, involving over 40 academic institutions around the world, we have tried to tackle the contentious question in the biggest study of its kind. As so often in science the answers turn out to be far more complicated than imagined. The study, published online today in Nature reveals that neither climate nor humans alone can account for the Ice Age mass extinctions. Using ancient megafauna DNA, climate data and the archaeological record, the findings indicate dramatically different responses of Ice Age species to climate change and humans. For example, the study shows that humans played no part in the extinction of the woolly rhino or the musk ox i Eurasia and that their demise can be entirely explained by climate change. On the other hand, humans aren’t off the hook when it comes to the extinction of the wild horse and the bison in Siberia. Our ancestors share responsibility for the megafauna extinctions with climate change. While the reindeer remain relatively unaffected by any of these factors, the causes of the extinction of the mammoths is still a mystery. Despite the unparalleled amount of data analysed in this study, we find no clear pattern distinguishing species that went extinct from species that survived, suggesting that it will be extremely challenging for experts to predict how existing mammals will respond to future global climate change.</p>

<p style="margin-top: -25px; text-align:right; color:lightgrey; "><i> &#8212; Dec. 29. 2009</i></p>
