<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Ideas / notes – Title of your project</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../notes/mbg-exercises.html" rel="next">
<link href="../notes/hybrid-incomp.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/ideas.html"><span class="chapter-title">Ideas / notes</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">Title of your project</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Project</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Notes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/chromatin.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/funding.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/hybrid-incomp.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Hybrid incomp</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/ideas.html" class="sidebar-item-text sidebar-link active"><span class="chapter-title">Ideas / notes</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/mbg-exercises.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">MBG exercises</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/meiotic-drive.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Meiotic drive</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/notes.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/phasetype.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Phase-type dist</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/prdm9.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">PRDM9</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/sex-antagonistic.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/spermatogenesis.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Spermatogenesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/strategy.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Research strategy</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/x-brain.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">X-brain</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/xi-msci.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Xi and MSCI</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul class="collapse">
  <li><a href="#x-brain" id="toc-x-brain" class="nav-link active" data-scroll-target="#x-brain">X-brain</a></li>
  <li><a href="#primate-diversity" id="toc-primate-diversity" class="nav-link" data-scroll-target="#primate-diversity">Primate diversity</a></li>
  <li><a href="#lewontins-paradox" id="toc-lewontins-paradox" class="nav-link" data-scroll-target="#lewontins-paradox">Lewontin’s paradox</a></li>
  <li><a href="#xy-drive-info-for-adding-to-notesa" id="toc-xy-drive-info-for-adding-to-notesa" class="nav-link" data-scroll-target="#xy-drive-info-for-adding-to-notesa">XY-drive info for adding to notesa</a></li>
  <li><a href="#xy-drive" id="toc-xy-drive" class="nav-link" data-scroll-target="#xy-drive">XY-drive</a></li>
  <li><a href="#mitochondria" id="toc-mitochondria" class="nav-link" data-scroll-target="#mitochondria">Mitochondria</a></li>
  <li><a href="#xy-deg-msci-xi" id="toc-xy-deg-msci-xi" class="nav-link" data-scroll-target="#xy-deg-msci-xi">XY-DEG-MSCI-XI</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/ideas.html"><span class="chapter-title">Ideas / notes</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-title">Ideas / notes</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<ul>
<li>Do species converge to a similar size determined mainly by the rate of reticulation. What amount of divergence should be consdidered one reticulating “super-species” for Ne to not increase backwards in time? 50.000 generations? What does it mean to drive on X that (and possible autosome compensation)<br>
</li>
<li>that Y is lost?<br>
</li>
<li>Natural experiment: Learn what it means to mole-rats (Ellobius lutescens) ) have lost the Y chromosome so they are X0, compared to closely related (Ellobius tancrei) that have XX.<br>
</li>
<li>gBGC and sex ratio Hypothesis:<br>
</li>
<li>The different contributions of transitions and transversions to GC* between species groups results from differences in the relative recombination rates in males and females. Transversions show stronger gBGC in males whereas transitions show higher bias in females.<br>
</li>
<li>Approach:<br>
</li>
<li>Construct ancestral recombination maps from ILS segment breaks and use the maps. Use known patterns of recombinaton rate ratio between sexes to derive a recombination rate for each sex. Compare observed GC*_transversion/GC*_transition to expectation computed from estimated male and female recombination rates.<br>
</li>
<li>Perspective:<br>
</li>
<li>If the hypothesis is true, gBGC patterns could be used to predict male/female relative map length in ancestral and extant species where recombination rate cannot/is not estimated. Could provide information to understand the basis of the Haldane-Huxley rule.<br>
</li>
<li>NB: Better idea to first compare stable hotspots in birds and dogs<br>
</li>
<li>chr17 inversion<br>
</li>
<li>deCODE Nature genetics paper: https://www.nature.com/articles/ng1508<br>
</li>
<li>Does it have a role in drive?<br>
</li>
<li>The H1/H2 polumorphism is apparently 3 mill years old<br>
</li>
<li>No recombination between them<br>
</li>
<li>The H2 has 20% freq in Europeans but is rate in Africa and almost absent in Asia<br>
</li>
<li>At the time of writing they couldl not determine which is ancestral<br>
</li>
<li>Islandic H2 females have more children and higher recombination rate<br>
</li>
<li>Contains two genes<br>
</li>
<li>**MAPT (Microtubuli Associated Protein Tau)**<br>
</li>
<li>MAPT gene mutations have been associated with several neurodegenerative disorders such as Alzheimer’s disease, Pick’s disease, frontotemporal dementia, cortico-basal degeneration and progressive supranuclear palsy. Promotes microtubule assembly and stability, and might be involved in the establishment and maintenance of neuronal polarity.<br>
</li>
<li>**NB:**In mice (made to have autism traits), reducing tau protein (MAPT) reduces or removes autism-like behaviour<br>
</li>
<li>**NSF**<br>
</li>
<li>Involved in intracellular protein transport<br>
</li>
<li>Located in Golgi apparatus; cytosol; and plasma membrane. Required for vesicle-mediated transport. Catalyzes the fusion of transport vesicles within the Golgi cisternae. Is also required for transport from the endoplasmic reticulum to the Golgi stack. Seems to function as a fusion protein required for the delivery of cargo proteins to all compartments of the Golgi stack independent of vesicle origin.<br>
</li>
<li>- Find out which one is ancestral<br>
</li>
<li>- Does Islanders have a Y chromosome diversity that is similarly unusual?<br>
</li>
<li><pre><code>Page 1 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>- What is the frequency of CENHAPS?<br>
</li>
<li>Linked selection: Elyashiv 2.0 Two closely releated species<br>
</li>
<li>Model positive selection seperately as function of Ne<br>
</li>
<li>Model BGS parameters shared between species<br>
</li>
<li>kaspermunch/DrosophilaLinkedSelectionMaps: The complete scripts that run the linked selection inference<br>
</li>
<li>Can Laurits find “population diagnostic” denisova segments…<br>
</li>
<li>- that can be used to see if there was indeed a “wave” from South east asia carrying the X drivers?<br>
</li>
<li>- if a segmet is almost fixed in a population, you can see if its MRCA in the population is close to 50,000. It would need to be to make sure it is not introduced from an other population later.<br>
</li>
<li>- To do that you can mask the archaic segments and date MRCA using RELATE<br>
</li>
<li>Use random forrest or gradient boosting to find a set of X SNPs that predict the Y haplogroup frequencies in human populations<br>
</li>
<li>Have I tried clustering DEG expression trajectories based on Xi variableXi and Xi-escape?<br>
</li>
<li>Maybe run Yuns model on chr7, chr8 and chrX to see if embeddings differ for chromatin motifs on chrX<br>
</li>
<li>List of Macaques from published primate diversity consortium (males/females),<br>
</li>
<li>range info<br>
</li>
<li>Indochina peninsula/Indonesia<br>
</li>
<li>- Macaca facicularis (long-tailed, crab-eating): 12/13, yes - Macaca_mulatta (rhesus): 7/0, yes<br>
</li>
<li>- Macaca_arctoides (stump-tailed): 3/3, yes<br>
</li>
<li>- Macaca_leonina (northern pig-tailed): 3/4, yes<br>
</li>
<li>- Macaca_nemestrina (southern pig-tailed): 1/4, yes Indonesia:<br>
</li>
<li>- Macaca_tonkeana: 1/0, no<br>
</li>
<li>- Macaca_nigra 0/1, no<br>
</li>
<li>- Macaca_siberu: 1/0, no<br>
</li>
<li>Indien/Nepal/Bhutan<br>
</li>
<li>- Macaca_assamensis: 1/1, yes Japan:<br>
</li>
<li>- Macaca_fuscata: 1/0, no Taiwan:<br>
</li>
<li>- Macaca_cyclopis: 1/0, no Tibet/Kina:<br>
</li>
<li>- Macaca_thibetana: 0/1, no Indien:<br>
</li>
<li>- Macaca radiata: 8/5, no<br>
</li>
<li>- Macaca_silenus 6/7, no<br>
</li>
<li>I think Jeff said there are macaques in USAID set from - Nepal<br>
</li>
<li>- Mianmar<br>
</li>
<li>- Vietnam - Taiwan<br>
</li>
<li>- Myanmar - Malaysia<br>
</li>
<li>Does prdm9 use relate to pop size? Is it a way to maintain recombination with very high population sizes?<br>
</li>
<li>Tom Gilbert about new passenger pigeon Project</li>
</ul>
<section id="x-brain" class="level2">
<h2 class="anchored" data-anchor-id="x-brain">X-brain</h2>
<ul>
<li>Student project Sweeps<br>
</li>
<li><pre><code>       Page 2 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>Look for papers about connection between epilepsy and autism emotional<br>
</li>
<li>regulation drugs<br>
</li>
<li>Questions for Karan Parker - Can I see the STAN code?<br>
</li>
<li>- A known mechanisms mainly biochemical or structural/developental?<br>
</li>
<li>- To what extent can synapsis issues (deficiency in neurotransmitters) be attributed to micotubule transport of tarnsmitters?<br>
</li>
<li>- How fine-grained is it possible to phenotypically autism subtypes?<br>
</li>
<li>- How much can be acribed to purely developmeltal timing?<br>
</li>
<li>Think about Anders Albrachtsens argument that Xi status does not matter and that all loci can be considdered additive<br>
</li>
<li>DeepRVAT and REGENIE<br>
</li>
<li>ARG-needle on Y chromosome GWAS<br>
</li>
<li>Sonic Hedgehog: a morphogen involved in axon guidance: R&amp;D Systems https://www.rndsystems.com/resources/articles/sonic-hedgehog-morphogen-involved-axon-guidance<br>
</li>
<li>Is the X activated and inactivated in each replication and if not, how does synapsis work for an X and an Xi<br>
</li>
<li>Is the missing heritability larger in males than females?<br>
</li>
<li>Make social scores for all individuals in Karen’s colony and array genotype all of them<br>
</li>
<li>Compute the effect on population-averaged polygenic risk score<br>
</li>
<li>Look for allele frequency differences between males and females and see if allele frequencies differ more between sexes on x than on autosomes<br>
</li>
<li>Identify allele frequency changes of SNPs driven by selection<br>
</li>
<li>Could we use random forest on ASD data to classify case and control, and then use the learned SNP interactions to infer non-additive interactions? Maybe females have less ASD because two copies of genes that variably escapes<br>
</li>
<li>Maybe there is much more cis interaction on the X if synteny is much more strongly conserved. Maybe i-LDSC will find higher heritability on X?<br>
</li>
<li>Are Xi genes enriched among sweep candidates?<br>
</li>
<li>Skewed XCI?<br>
</li>
<li>ASD genes affect different mechianisms, it is possible to have multiple Autisms. In that way it may be different to distringuish ASD subtypes if they are in fact combinatons of different condistions with different severity. Ideally we would do NFM on it…<br>
</li>
<li>Compute how much the population polygenic risk score for ASD increased when ECHs rose in frequency<br>
</li>
<li>Make venn figures with overlap between XCI / Matos_common / ECH / gametologs<br>
</li>
<li>Make figure with distributions of p-values for the same subsets<br>
</li>
<li>Plot p-values across the chomosomes somehow labelling the combinations of XCI, common, ECH, gametologs (colors, shape, etc.)<br>
</li>
<li>Do test of Fisher exact test of ASD in nDEG vs NPX genes AFTER removing all 75% EHC region genes (or low-ILS genes) from both sets<br>
</li>
<li><pre><code>                  Page 3 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21</li>
</ul>
</section>
<section id="primate-diversity" class="level2">
<h2 class="anchored" data-anchor-id="primate-diversity">Primate diversity</h2>
<ul>
<li>How does CV across all other species in a window explain X diversity in that<br>
</li>
<li>window?<br>
</li>
<li>How well does the CV in one species explain X/A in another<br>
</li>
<li>Stratify with sift score or cons and see if diversity in species with different X/A<br>
</li>
<li>ratios has different responses in each category. Or alternatively compute<br>
</li>
<li>interference_map scores with scalings corresponding to different time frames<br>
</li>
<li>(sweeps &lt;500 gens rate_scale=500, bgs &gt;500?… (depending on s of cuase)).<br>
</li>
<li>Is the “MBE smiley” plot higher on the right for the X chromosome?<br>
</li>
<li>The difference in global Ne between two species is made of up of the Ne<br>
</li>
<li>differences in 100kb windows. Each 100kb Ne can be explained globally by<br>
</li>
<li>census size, and locally by, phastcons, yun’s score, proximity to a gene,<br>
</li>
<li>recombination rate. Making a regression model could expose the connection<br>
</li>
<li>between census size and sweeps.<br>
</li>
<li>Are the residuals of a Ne vs Nc regression explained by recombination rate<br>
</li>
<li>differences among species?<br>
</li>
<li>Multivariate Gaussian Random Walk — PyMC example gallery https://www.pymc.io/projects/examples/en/latest/time_series/MvGaussianRandomWalk_demo.html<br>
</li>
<li>Modeling of multiple regression model with array form - Questions / v5 - PyMC<br>
</li>
<li>Discourse https://discourse.pymc.io/t/modeling-of-multiple-regression-model-with-array-form/12430<br>
</li>
<li>Make plan for maximizing value of Eriks primate work</li>
</ul>
</section>
<section id="lewontins-paradox" class="level2">
<h2 class="anchored" data-anchor-id="lewontins-paradox">Lewontin’s paradox</h2>
<ul>
<li>Neutral theory predicts that genetic diversity increases with population size, yet<br>
</li>
<li>observed levels of diversity across metazoans vary only two orders of magnitude while population sizes vary over several. This unexpectedly narrow range of diversity is known as Lewontin’s Paradox of Variation (1974). Quantifying the relationship between genetic diversity and population size suggests natural selection cannot explain Lewontin’s Paradox https://doi.org/10.7554/eLife.67509<br>
</li>
<li>Does reproductive variance (in fertility/even meiosis) correlate with census size? A larger N means more frequent introduction of new (potentially selected) variants. If larger N means more stable alleles which again allow alleles to be more extreme drivers while still be averagely even in meiosis… Then each the variance across idividual X/Y pairs would be higher. See if Lewinton’s paradox applies to species without x chromosomes… See if the variance in the allele drive values is larger for higher N. Remember to extract only odd rows and even columns. Distortion does not in it self does not produce a fitness reduction by more than 0.5. It will have to include an decrease in fertility too. If there are a very large number of alleles, then there cannot emerge a X new allele that is equally fertile with all the Y chromosomes. It can only be averagely<br>
</li>
<li><pre><code>        Page 4 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>more fertile by fitting with as many as possible.<br>
</li>
<li>The dynamic balance of antagonizing drivers of X and Y is affected by census size and/or exponential population growth in a way that induces a larger genetic load on X and Y in populations with very large census sizes<br>
</li>
<li>Can reversed sweeps explain even Ne across species?<br>
</li>
<li>Can the same rate of environment change across species homogenize Ne if we assume positive selection is mutation limited and thus proportional to census size?<br>
</li>
<li>Do species with larger Ne tend to have smaller X/A?<br>
</li>
<li>If we know DFEs and Mb-scale recombination rate is shared among closely related species, then 1Mb windows should be comparable between species, because each window average over the differences and adjacent windows have similar mean rates in the two species.<br>
</li>
<li>Rolling regression using pymc? - comparing pi of two closely related species with different Ne along the genome<br>
</li>
<li>Interferrence score?<br>
</li>
<li>Fit a combination of interferrence_maps across different scales to diversity data. If high Nc species do not have as high Ne as expected, what scale of linker selection explains this. Near or far?<br>
</li>
<li>Run Yuns model on the primates to get constrained sites<br>
</li>
<li>Make rolling regression of pi explained by pi of other species and rec rate With many genomes from two closely related species, would it make sense to look at relate trees rather than diversity?<br>
</li>
<li>Could my run statistic quantify selection on quantitative traits across species with different Ne?<br>
</li>
<li>Do ILS smiley plot as rolling regression<br>
</li>
<li>Ask Thomas about Guy Sella shift phase and Lewintons paradox<br>
</li>
<li>Can my run stat quantify selection on quantitative traits by rapid changes to allele frequencies a la Guy Sella<br>
</li>
<li>If we assume that functional sites and recombination rate is the same in band- tailed and passenger pigeons, then we can subtract the pi on the two in each window to get effect of only population size. What would then be the density and strength of sweeps required to explain the strong edge effect?<br>
</li>
<li>Huge population sizes (maybe like passenger pigeon) poses problem for synapsis and recombination, leaving recombination to conserved regions like exons</li>
</ul>
</section>
<section id="xy-drive-info-for-adding-to-notesa" class="level2">
<h2 class="anchored" data-anchor-id="xy-drive-info-for-adding-to-notesa">XY-drive info for adding to notesa</h2>
<ul>
<li>Tissue expression of ACTRT2 - the most enriched gene in testis!! https://www.proteinatlas.org/ENSG00000169717-ACTRT2/tissue<br>
</li>
<li>https://www.proteinatlas.org/ENSG00000142700-DMRTA2<br>
</li>
<li>Genes and brain malformations associated with abnormal neuron positioning |<br>
</li>
<li>Molecular Brain | Full Text https://molecularbrain.biomedcentral.com/articles/10.1186/s13041-015-0164-4#Tab1<br>
</li>
<li>A male germ-cell-specific ribosome controls male fertility | Nature<br>
</li>
<li><pre><code>                Page 5 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>https://www.nature.com/articles/s41586-022-05508-0<br>
</li>
<li>Macro-haplogroup L (mtDNA) - Wikipedia https://en.wikipedia.org/wiki/Macro-haplogroup_L_%28mtDNA%29<br>
</li>
<li>DMRTA2 Gene - GeneCards | DMTA2 Protein | DMTA2 Antibody https://www.genecards.org/cgi-bin/carddisp.pl?gene=DMRTA2<br>
</li>
<li>Insersect Cillary genes with spermatogenesis expression categories https://www.readcube.com/library/ff394a98-20f7-4882-871c- 840e9f73d65b:932c2460-f897-46ba-b3f1-76358eaf2077 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6484137/ https://www.sciencedirect.com/science/article/pii/S1055790308002108 https://www.sciencedirect.com/science/article/pii/S1055790309000438#fig1 https://royalsocietypublishing.org/doi/full/10.1098/rsbl.2009.0741<br>
</li>
<li>The human proteome in testis - The Human Protein Atlas https://www.proteinatlas.org/humanproteome/tissue/testis<br>
</li>
<li>Global Developmental Gene Expression and Pathway Analysis of Normal Brain<br>
</li>
<li>Development and Mouse Models of Human Neuronal Migration Defects https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053345/<br>
</li>
<li>» How big are mitochondria? http://book.bionumbers.org/how-big-are-mitochondria/<br>
</li>
<li>Subcellular - DMRTA2 - The Human Protein Atlas https://www.proteinatlas.org/ENSG00000142700-DMRTA2/subcellular#<br>
</li>
<li>ZNF711 Gene - GeneCards | ZN711 Protein | ZN711 Antibody https://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNF711<br>
</li>
<li>Mechanisms of meiotic drive in symmetric and asymmetric meiosis<br>
</li>
<li>Genetic basis of acephalic spermatozoa syndrome, and intracytoplasmic<br>
</li>
<li>sperm injection outcomes in infertile men: a systematic scoping review https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:1609b585-8304-4567-a99a-<br>
</li>
<li>6a2fc255afc6<br>
</li>
<li>Reconsider F8 as potential driver. It is in pages new paper. Check if it is also<br>
</li>
<li>det<br>
</li>
<li>H2AX Gene - GeneCards | H2AX Protein | H2AX Antibody https://www.genecards.org/cgi-bin/carddisp.pl?gene=H2AX#summaries<br>
</li>
<li>Circular RNAs: Novel Biomarkers in Spermatogenesis Defects and Male<br>
</li>
<li>Infertility | SpringerLink https://link.springer.com/article/10.1007/s43032-022-00885-3<br>
</li>
<li>Loss of circSRY reduces γH2AX level in germ cells and impairs mouse spermatogenesis - PubMed<br>
</li>
<li>https://pubmed.ncbi.nlm.nih.gov/36414375/<br>
</li>
<li>CircSry regulates spermatogenesis by enhancing γH2AX expression via sponging miR-138-5p | bioRxiv<br>
</li>
<li>https://www.biorxiv.org/content/10.1101/2021.10.25.465684v1<br>
</li>
<li>Zhao: Targeting mitochondria-located circRNA SCAR… - Google Scholar https://scholar.google.com/scholar_lookup?title=Targeting%20Mitochondria-<br>
</li>
<li>Located%20circRNA%20SCAR%20Alleviates%20NASH%20via%20Reducing%20mROS%20Output&amp;author=Q.Y.%20Zha 93#d=gs_qabs&amp;t=1677751950808&amp;u=%23p%3DgRHwBM8CkhYJ<br>
</li>
<li><pre><code>                  Page 6 of 15  </code></pre></li>
<li>o&amp;publication<br>
</li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>Wnt pathway https://en.wikipedia.org/wiki/Wnt_signaling_pathway?wprov=sfti1<br>
</li>
<li>Wnt signaling pathway - Wikipedia https://en.m.wikipedia.org/wiki/Wnt_signaling_pathway<br>
</li>
<li>Akap4<br>
</li>
<li>GO:0097224: sperm connecting piece<br>
</li>
<li>The segment of the sperm flagellum that attaches to the implantation fossa of the nucleus in the sperm head; from the remnant of the centriole at this point, the axoneme extends throughout the length of the flagellum.<br>
</li>
<li>GO:0097225: sperm midpiece<br>
</li>
<li>The highly organized segment of the sperm flagellum which begins at the connecting piece and is characterized by the presence of 9 outer dense fibers (ODFs) that lie outside each of the 9 outer axonemal microtubule doublets and by a sheath of mitochondria that encloses the ODFs and the axoneme; the midpiece terminates about one-fourth of the way down the sperm flagellum at the annulus, which marks the beginning of the principal piece.<br>
</li>
<li>CFAP47 sperm axoneme assembly GO:0007288: sperm axoneme assembly<br>
</li>
<li>The assembly and organization of the sperm flagellar axoneme, the bundle of microtubules and associated proteins that forms the core of the eukaryotic sperm flagellum, and is responsible for movement.<br>
</li>
<li>GO:0097224: sperm connecting piece<br>
</li>
<li>The segment of the sperm flagellum that attaches to the implantation fossa of the nucleus in the sperm head; from the remnant of the centriole at this point, the axoneme extends throughout the length of the flagellum.<br>
</li>
<li>CNTLN Gene - GeneCards | CNTLN Protein | CNTLN Antibody https://www.genecards.org/cgi-bin/carddisp.pl?gene=CNTLN<br>
</li>
<li>CENTLEIN is localized at the sperm head–tail coupling apparatus a… |<br>
</li>
<li>Download Scientific Diagram CENTLEIN<br>
</li>
<li>https://www.researchgate.net/figure/CENTLEIN-is-localized-at-the-sperm-head-tail-coupling-apparatus-a- CENTLEIN-was_fig7_353885254<br>
</li>
<li>https://app.readcube.com/library/ff394a98-20f7-4882-871c- 840e9f73d65b/all/(sidepanel:details)?item_id=85c3d143-9b8d-4125-8467- 8ea2d7024e43&amp;collection_id=ff394a98-20f7-4882-871c-840e9f73d65b<br>
</li>
<li>The Renin (REN) promoter is upregulated by SRY and downregulated by SOX3, likely through trans and cis mechanisms, respectively.<br>
</li>
<li>Frontiers | The Functions, Methods, and Mobility of Mitochondrial Transfer<br>
</li>
<li>Between Cells https://www.frontiersin.org/articles/10.3389/fonc.2021.672781/full<br>
</li>
<li>CircDNA paper <a href="https://www.cell.com/cell/pdf/S0092-8674(20)31225-3.pdf">https://www.cell.com/cell/pdf/S0092-8674(20)31225-3.pdf</a><br>
</li>
<li></li>
</ul>
</section>
<section id="xy-drive" class="level2">
<h2 class="anchored" data-anchor-id="xy-drive">XY-drive</h2>
<ul>
<li>Get complete list of all XY homologs incl.&nbsp;HSFX<br>
</li>
<li>Expression of all genes with homologous on y<br>
</li>
<li>Why are XY homologous better candidates for drivers: they can do the same thing in two different ways, and they are DEG<br>
</li>
<li>XCI - MSCI link… Maybe example of compensation: Autosomal genetic variation is associated with DNA methylation in regions variably escaping X- chromosome inactivation<br>
</li>
<li><pre><code>         Page 7 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:618b6abb-9f98-4669-bce1- c98623aa5743<br>
</li>
<li>Do more of gene with homologs on X and Y have antisense transcripts?<br>
</li>
<li>If the repeat numbers in the fragile X loci modulate expression, then that could<br>
</li>
<li>be what constitutes the drive…<br>
</li>
<li>Find all genes (antisense transcripts) that ends with “-AS\d+$”, and see if<br>
</li>
<li>genes they are antisense to are any particular kinds…<br>
</li>
<li>Are bantu and west adrican pygmies in different mt clades? Look at chr 3 to<br>
</li>
<li>region in baboons<br>
</li>
<li>The chromatoid body in spermatogenesis - PARVINEN - 2005 - International<br>
</li>
<li>Journal of Andrology - Wiley Online Library https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-2605.2005.00542.x<br>
</li>
<li>Brain tissue expression of AGBL4 - Summary - The Human Protein Atlas https://www.proteinatlas.org/ENSG00000186094-AGBL4/brain<br>
</li>
<li>Redo the peaks notebook with all your new knowledge and Meritxells data Revisit peaks notebook and look for hints about mitochondria<br>
</li>
<li>Maybe Olive MT stopped its introgression at Utzungwa because of the baboon interaction with Kipunji somehow? Maybe the exposure to Kipunji gave the baboons there some standing variation that could resist the invasion?<br>
</li>
<li>Could meiotic drive explain why Neandertal ancestry is so sparse on the X<br>
</li>
<li>chromosome? https://johnhawks.net/weblog/positive-selection-x-chromosomes-recent-human-evolution/<br>
</li>
<li>What is the overlap between XCI genes and MSCI genes? Are DE genes special in that regard?<br>
</li>
<li>Could it be that nuclear control of MT came about becuse male-specifc compensating selection does not work in MT (because the are not passed on from males)<br>
</li>
<li>Dynein ATPase - an overview | ScienceDirect Topics https://www.sciencedirect.com/topics/neuroscience/dynein-atpase<br>
</li>
<li>List of variants in gene combination ATP6AP2, BCOR, CFAP47, CXorf22, CXorf30, CXorf38, CYBB, DMD, DYNLT3, FAM47A, FAM47B, FAM47C, FTHL17, GK, H2AP, LANCL3, MAGEB16, MED14, MID1IP1, MPC1L, NR0B1, OTC, PRRG1, RPGR, SRPX, SYTL5, TAB3, TASL, TMEM47, TSPAN7, USP9X, XK reported as pathogenic for Ornithine carbamoyltransferase deficiency -<br>
</li>
<li>Admixture Baboons MT<br>
</li>
<li><pre><code>           ClinVar Miner  </code></pre></li>
<li>https://clinvarminer.genetics.utah.edu/variants-by- condition/Ornithine%20carbamoyltransferase%20deficiency/gene/ATP6AP2%2C%20BCOR%2C%20CFAP47%2C%20CX<br>
</li>
<li>Could argweaver have trouble finding introggressions if there is a sweep in humans?<br>
</li>
<li>Circular RNAs: biogenesis, expression and their potential roles in reproduction<br>
</li>
<li>- PMC<br>
</li>
<li>Circular RNAs: biogenesis, expression and their potential roles in reproduction - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5773157/#!po=30.8140<br>
</li>
<li>Loss of circSRY reduces γH2AX level in germ cells and impairs mouse<br>
</li>
<li>spermatogenesis - PubMed<br>
</li>
<li>Loss of circSRY reduces γH2AX level in germ cells and impairs mouse spermatogenesis - PubMed<br>
</li>
<li>Page 8 of 15<br>
</li>
<li>orf22%2C%2<br>
</li>
<li>0<br>
</li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>https://pubmed.ncbi.nlm.nih.gov/36414375/<br>
</li>
<li>Can drive help a species jump a fitness peak?<br>
</li>
<li>Pathogenic Variants in ACTRT1 Cause Acephalic Spermatozoa Syndrome https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:85c3d143-9b8d-4125-8467- 8ea2d7024e43<br>
</li>
<li>Look for high Neanderthal introgression into humans<br>
</li>
<li>Cytoplasmic dynein regulates the subcellular distribution of mitochondria by<br>
</li>
<li>controlling the recruitment of the fission factor dynamin-related protein-1 https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:7d8371d6-231d-4704-8381- e577dd977c33<br>
</li>
<li>Skriv til Hubisz<br>
</li>
<li>How does global X/A ILS/Ne correspond to ILS/pi away from genes?<br>
</li>
<li>Heterozygosity with distance from genes on x and auto. Use Elyashiv 2018<br>
</li>
<li>Model BDMI inkomp selection between chr3 and MT<br>
</li>
<li>Duplicate ILS pipeline to make windowed admixture proportions from Moi’s<br>
</li>
<li>data<br>
</li>
<li>path to an individual’s fragment example:<br>
</li>
<li>/home/moicoll/GenerationInterval/people/moi/sandbox/dec/1KGP/HG02773.new.txt<br>
</li>
<li>Cadherins and neuropsychiatric disorders - PubMed https://pubmed.ncbi.nlm.nih.gov/22765916/<br>
</li>
<li>Cadhedrines in Autism https://studenttheses.uu.nl/bitstream/handle/20.500.12932/12261/Sophie%20-%20Master%20Thesis%20- %20Final%20version.pdf?sequence=1<br>
</li>
<li>Selection in driver dynamics be positive selection on multiple alelles at the same time<br>
</li>
<li>Loss of perinuclear theca ACTRT1 causes acrosome detachment and severe male subfertility in mice - PubMed<br>
</li>
<li>https://pubmed.ncbi.nlm.nih.gov/35616329/<br>
</li>
<li>Pathogenic Variants in ACTRT1 Cause Acephalic Spermatozoa Syndrome https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8377740/<br>
</li>
<li>TKTL1 Gene - GeneCards | TKTL1 Protein | TKTL1 Antibody https://www.genecards.org/cgi-bin/carddisp.pl?gene=TKTL1<br>
</li>
<li>Make a high-ILS track like the low-ILS track and see if they overlap more on<br>
</li>
<li>the X than on the autosomes<br>
</li>
<li>Functional significance of the sex chromosomes during spermatogenesis https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:932c2460-f897-46ba-b3f1- 76358eaf2077<br>
</li>
<li>Bridging the gap between the evolutionary dynamics and the molecular<br>
</li>
<li>mechanisms of meiosis: a model based exploration of the PRDM9 intra-<br>
</li>
<li>genomic Red Queen | bioRxiv https://www.biorxiv.org/content/10.1101/2023.03.08.531712v1<br>
</li>
<li>The Role of Methylation in Gene Expression | Learn Science at Scitable https://www.nature.com/scitable/topicpage/the-role-of-methylation-in-gene-expression-1070/<br>
</li>
<li>What is the required ratio between increased drive s and lowered fitness s for driver to be selected?<br>
</li>
<li>Found out many of the candidate genes have indels that span codons<br>
</li>
<li>Admixture ILS<br>
</li>
<li><pre><code>                  Page 9 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>(presumably producing a new amino acid). They are filtered out when requiring all codons to be a multiples of inframe triplets. Requiring just that they are multiples of inframe triplets they are included. Even with this less strict filtering some interesting genes are discarded: bla. AFF2, ZCCHC13 , USP9X, RLIM, VCX3A, HSFX1, RBMXL3, FANCB, VCX, and a lot of the XY homologs on Y: USP9Y, RBMY1F, HSFY2, TGIF2LY, and TSPY genes<br>
</li>
<li>Is ikers high ILS in low species true signal?<br>
</li>
<li>Do codeml genes have any particular expression patterns in spermatogenesis? DEG - Do k-means expression clustering and barplot of codeml genes and the rest<br>
</li>
<li>Are codeml genes enriched for DE genes?<br>
</li>
<li>Lav kæmpe A3 figur med hele chrX i XY-drive projekt som du kan tilføje<br>
</li>
<li>subpanels til<br>
</li>
<li>Do drivers / differentially expressed genes have more structural variation that<br>
</li>
<li>could affect their function/expression?<br>
</li>
<li>There is a lot of rare variation on AFF2, associated with ASD: Excess variants in<br>
</li>
<li>AFF2 detected by massively parallel sequencing of males with autism<br>
</li>
<li>spectrum disorder - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441129/<br>
</li>
<li>Flex sweep on x<br>
</li>
<li>Dynlt3 associaties with Bub3 and spindle meiotic spindle checkpoint complex<br>
</li>
<li>Check if any of the candidate genes could be relalted to the ribosome adn<br>
</li>
<li>potentially to the newly found RibosomeST<br>
</li>
<li>Why would a gene move off X if it is not a driver, and why can it if it competes<br>
</li>
<li>against homologs on Y (without inducing the oposite drive). I guess mainly if<br>
</li>
<li>drive is a competition between X and autosomes<br>
</li>
<li>Evolutionary quantitative genetics of sperm - ScienceDirect https://www.sciencedirect.com/science/article/pii/B9780123725684000100<br>
</li>
<li>Is the subset of MSCI escapers that are DE more represented among XCI escapers? (or maybe among the subset that is deltaE?)<br>
</li>
<li>Is it always the heterogametic sex that introgresses more?<br>
</li>
<li>Wiring through tunneling nanotubes – from electrical signals to organelle<br>
</li>
<li>transfer | Journal of Cell Science | The Company of Biologists https://journals.biologists.com/jcs/article/125/5/1089/33229/Wiring-through-tunneling-nanotubes-from- electrical<br>
</li>
<li>Look at heterozygosity in Altai and Vindia<br>
</li>
<li>Are vindia and Altai males or females? ChrX phaseing<br>
</li>
<li>Centromeres try to get pulled. There are proteins that can work against. Do centromers and proteins need to be linked like on the x and y for this to work? Check genes in Altai genome paper: “A high-coverage Neandertal genome from Chagyrskaya Cave” “We find that genes highly expressed in the striatum in the basal ganglia of the brain carry more amino acid-changing substitutions than genes expressed elsewhere in the brain, suggesting that the striatum may have evolved unique functions in Neandertals.” Are they expressed in testis? Make list of all HN genes with evidence of adaptive introgression. Relation to CENHAPS or XY drive?<br>
</li>
<li>MSCI XCI<br>
</li>
<li><pre><code>             Page 10 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>Erik about running RELATE or do it myself<br>
</li>
<li>Can we use COLATE to look for ECHs in Neanderthals?<br>
</li>
<li>How many brain genes and how many sfari genes?<br>
</li>
<li>Are DEG genes enriched for ASD or is it only the subset with evidence of selection?<br>
</li>
<li>Find a way to match genes by selection trajectory so that I can make valid background sets for enrichment tests<br>
</li>
<li>SFARI | Gene expression patterns that underlie functional brain activity are<br>
</li>
<li>disrupted in autism https://www.sfari.org/2022/08/29/gene-expression-patterns-that-underlie-functional-brain-activity-are- disrupted-in-autism/<br>
</li>
<li>Homo sapiens (human) non-protein coding lnc-CFAP47-2:1 | URS00008B57AC https://rnacentral.org/rna/URS00008B57AC/9606<br>
</li>
<li>Deleterious variants in X-linked &lt;em&gt;CFAP47 &lt;/em&gt;induce<br>
</li>
<li>asthenoteratozoospermia and primary male infertility - ePrints - Newcastle<br>
</li>
<li>University https://eprints.ncl.ac.uk/273329<br>
</li>
<li>WDR87 interacts with CFAP47 protein in the middle piece of spermatozoa<br>
</li>
<li>flagella to participate in sperm tail assembly | Molecular Human Reproduction |<br>
</li>
<li>Oxford Academic https://academic.oup.com/molehr/article-abstract/29/1/gaac042/6960929?login=false<br>
</li>
<li>FMR1 and meiosis in females<br>
</li>
<li>Do a custom codeml analysis of DYNLT3 including only great apes<br>
</li>
<li>The dynamics of gene expression during and post meiosis sets the sperm<br>
</li>
<li>agenda - Pandey - 2019 - Molecular Reproduction and Development - Wiley<br>
</li>
<li>Online Library https://onlinelibrary.wiley.com/doi/10.1002/mrd.23278<br>
</li>
<li>Overlap between XI and MSCI genes<br>
</li>
<li>RNA stability controlled by m6A methylation contributes to X-to-autosome<br>
</li>
<li>dosage compensation in mammals | Nature Structural &amp; Molecular Biology https://www.nature.com/articles/s41594-023-00997-7<br>
</li>
<li>Loss of ATRX leads to chromosome cohesion and congression defects - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2213576/<br>
</li>
<li>A novel exomal ATRX mutation with preferential transmission to offspring: A<br>
</li>
<li>case report and review of the literature for transmission ratio distortion in ATRX<br>
</li>
<li>families - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7646821/<br>
</li>
<li>Has ref to MAOA: Neuronal signaling repertoire in the mammalian sperm<br>
</li>
<li>functionality | Biology of Reproduction | Oxford Academic https://academic.oup.com/biolreprod/article/96/3/505/3038274?login=false<br>
</li>
<li>CatSper: A Unique Calcium Channel of the Sperm Flagellum - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5914511/<br>
</li>
<li>Evolutionary Genomics Reveals Lineage-Specific Gene Loss and Rapid<br>
</li>
<li>Evolution of a Sperm-Specific Ion Channel Complex: CatSpers and CatSperβ -<br>
</li>
<li>PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572835/<br>
</li>
<li><pre><code>               Page 11 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>Does the mouse T haplotype drive because of the Dyneins on it (all light chain!)<br>
</li>
<li>Direct regulation of Chk1 protein stability by E3 ubiquitin ligase HUWE1. - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7226928/<br>
</li>
<li>Direct regulation of Chk1 protein stability by E3 ubiquitin ligase HUWE1. - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7226928/<br>
</li>
<li>Could the Robertsonian translocation creating the human chr2 be what<br>
</li>
<li>triggered the extensive low-ILS on chrX and chr3? Could explain why we see<br>
</li>
<li>so much here anot not so much in other ancestral species. Are there other<br>
</li>
<li>similar translocations in other ancestral species?<br>
</li>
<li>Do metacentric chromodsomes go pericentric over time?<br>
</li>
<li>Are metacentric chromosomes always old pericentric ones?<br>
</li>
<li>Think about NMF for y chromosome kmer profiles…<br>
</li>
<li>Background and arguments for X-brain https://docs.google.com/document/d/1mTRLDXrmR0WVcXv-vDvLryNRLFdAhIuP9hr1rCGdS0o/edit<br>
</li>
<li>Meritxell om hvorfor CFAB47 er faldet ud<br>
</li>
<li>add cytobands to geneinfo ideograms<br>
</li>
<li>Update my peak candidates using Meritxells new version of cDEGs<br>
</li>
<li>Make a new clean version of nb_20-updated_sweep_enrichments.ipynb in xy-<br>
</li>
<li>drive<br>
</li>
<li>Are the peaks on the borders mostly the cDEG ones?<br>
</li>
<li>Why do genes with a Y gametolog more commonly escape XI? Check what<br>
</li>
<li>proportion of them do, and if all XI escapers are enriched for X/Y gametologs<br>
</li>
<li>Genes that Escape X-Inactivation in Humans Have High Intraspecific Variability<br>
</li>
<li>in Expression, Are Associated with Mental Impairment but Are Not Slow<br>
</li>
<li>Evolving | Molecular Biology and Evolution | Oxford Academic https://academic.oup.com/mbe/article/33/1/302/2579282<br>
</li>
<li>Læse CENHAPS 2.0 artikel<br>
</li>
<li>Are ampliconic genes on the other side of the borders (in the white regions)? Is<br>
</li>
<li>that why ECHs are proximal to ampli<br>
</li>
<li>Are DEG genes enriched on the inner borders of dark g-bands? or only<br>
</li>
<li>selected DEGs?<br>
</li>
<li>Look for CENHAPS introgression<br>
</li>
<li>Can individuals be grouped by their ECHs in a way that matches the thier<br>
</li>
<li>CENHAP?<br>
</li>
<li>The DYNLT3 Light Chain Directly Links Cytoplasmic Dynein to a Spindle<br>
</li>
<li>Checkpoint Protein, Bub3 - ScienceDirect https://www.sciencedirect.com/science/article/pii/S0021925820766998<br>
</li>
<li>Loss of ATRX leads to chromosome cohesion and congression defects |<br>
</li>
<li>Journal of Cell Biology | Rockefeller University Press https://rupress.org/jcb/article/180/2/315/45187/Loss-of-ATRX-leads-to-chromosome-cohesion-and<br>
</li>
<li>Positioning to get out of meiosis: the asymmetry of division | Human<br>
</li>
<li>Reproduction Update | Oxford Academic https://academic.oup.com/humupd/article/17/1/68/639813<br>
</li>
<li>Test overlap between genes associated with neuron migration + sperm flagellum, or neuronal migration + postmeiotic expression<br>
</li>
<li><pre><code>                    Page 12 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>Is GWAS variance on height related to positive selection? - It could be compensating selection to counter drive.<br>
</li>
<li>- The argument would be that drive affects the integrity of a large number of pathways together affecting cell growth and or mitosis speed/fidelity.<br>
</li>
<li>- If true, then the GWAS variance should overlap both ILS/selection<br>
</li>
<li>- It does not necessarily only affect sperm expressed genes, because compensating genes could share pathways with drivers but only be expressed in the detrimentally affected tissues.<br>
</li>
<li>Get gene list from this paper: Systematic analysis and prediction of genes<br>
</li>
<li>associated with monogenic disorders on human chromosome X | Nature<br>
</li>
<li>Communications https://www.nature.com/articles/s41467-022-34264-y<br>
</li>
<li>Remind meritxell to do power analysis to see if she can also make a list that is<br>
</li>
<li>at most some amount of diff. exp.<br>
</li>
<li>Encyclopedia of Cell Biology https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:3dbd8369-8a79-46ad-8b30- 75ca9ab8f031<br>
</li>
<li>Prøv Laurits HMM men hun med den ene af de to afrikanske kalder adskilt af 200.000 år. Måske kan det finde HN admix som senere er NH admix<br>
</li>
<li>Non syn or sift snp frequences in ECH regions vs changes on ECH vs remaining chrimosome (controlling for lower Ne and Rec)<br>
</li>
<li>A novel acrosomal protein, IQCF1, involved in sperm capacitation and the<br>
</li>
<li>acrosome reaction - PubMed https://pubmed.ncbi.nlm.nih.gov/25380116/<br>
</li>
<li>It looks like ampliconic/multicopy genes are missing from the XI status gene lists (and maybe more lists?)<br>
</li>
<li>What if the deltaE effect works like the influence from Y? With an additional X there is one more CENHAPS. See if deltaE genes are on heterochromatin borders<br>
</li>
<li>Tjek om det svenske arvelighed studie tager højde for at y kromosomers effekt. X Y interaktion<br>
</li>
<li>Aurora about marsupial interchrom interaction and perternal XI<br>
</li>
<li>Create seperate olive and hamadryas chromatin maps<br>
</li>
<li>Du escape could be a way that x (or the female organism) remains in control of a gene).<br>
</li>
<li>Brush off drive simulations for David<br>
</li>
<li>Quarto In xy-drive and subprojects<br>
</li>
<li>Read “The human Y and inactive X chromosomes similarly modulate autosomal gene expression”. If human Y really modulate autosomal expression like in drosophila that that is add odds with it being the same as Xi, if that is what th epaper says…</li>
</ul>
</section>
<section id="mitochondria" class="level2">
<h2 class="anchored" data-anchor-id="mitochondria">Mitochondria</h2>
<ul>
<li>Association Between Mitochondrial DNA Haplogroup Variation and Autism<br>
</li>
<li>Spectrum Disorders<br>
</li>
<li>https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:44f69fec-2411-413d-8a45- 51cade2d70f1<br>
</li>
<li>chr3<br>
</li>
<li><pre><code>       Chromatin  </code></pre></li>
<li>Focus Work<br>
</li>
<li><pre><code>   Page 13 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>How will MT respond to drive if it is only maternally inherited?<br>
</li>
<li>Human mitochondrial DNA haplogroup - Wikipedia https://en.wikipedia.org/wiki/Human_mitochondrial_DNA_haplogroup<br>
</li>
<li>Association Between Mitochondrial DNA Haplogroup Variation and Autism<br>
</li>
<li>Spectrum Disorders | Autism Spectrum Disorders | JAMA Psychiatry | JAMA<br>
</li>
<li>Network https://jamanetwork.com/journals/jamapsychiatry/fullarticle/2649278<br>
</li>
<li>Biparental Inheritance of Mitochondrial DNA in Humans | PNAS https://www.pnas.org/doi/10.1073/pnas.1810946115<br>
</li>
<li>Haplotypes, mitochondria and autism - Mapping Ignorance https://mappingignorance.org/2019/01/07/haplotypes-mitochondria-and-autism/<br>
</li>
<li>Regulation of mitochondria-dynactin interaction and mitochondrial retrograde<br>
</li>
<li>transport in axons - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413347/<br>
</li>
<li>Mitochondrial localization, import, and mitochondrial function of the androgen<br>
</li>
<li>receptor - PMC<br>
</li>
<li>Mitochondrial localization, import, and mitochondrial function of the androgen receptor - PMC https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6484137/<br>
</li>
<li>Ask Dan if apoptosis could work in spermatids/spermatozoa<br>
</li>
<li>Control of mitochondrial transport and localization in neurons https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:e7c71d60-4a46-4ee1-a6d7- 9dd24289e80e<br>
</li>
<li>MT phylogenies for primates<br>
</li>
<li>Mitochondrial Dynamics — Mitochondrial Fission and Fusion in Human<br>
</li>
<li>Diseases | NEJM https://www.nejm.org/doi/10.1056/NEJMra1215233<br>
</li>
<li>Meiotic recombination mirrors patterns of germline replication in mice and<br>
</li>
<li>humans https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:c8286aa8-e22d-44c6-9021- 13342e6f5c41<br>
</li>
<li>A reassessment of explanations for discordant introgressions of mitochondrial<br>
</li>
<li>and nuclear genomes - Bonnet - 2017 - Evolution - Wiley Online Library https://onlinelibrary.wiley.com/doi/10.1111/evo.13296<br>
</li>
<li>Can Meritxell see if the higher Coi In X is due to more mitochondria or higher transcription?<br>
</li>
<li>Do mitochondria genes often have a pachytene peak or is that only CO1 CO2 and the ND genes that Mishmar finds?<br>
</li>
<li>Ask Mishmar about the potential for X regulation of mitochondrial expression Could the chr1 gene with only expression in spermatids be one that compensates x driven transport of mitochondria?<br>
</li>
<li>See if there is a GO for mitochondrial transport<br>
</li>
<li>Read mitochondria capture paper<br>
</li>
<li>XWAS extending Mishmars study on genes associated with MT gene<br>
</li>
<li>expression<br>
</li>
<li>- Read Mishmars paper<br>
</li>
<li>- Mail Dan Mishmar to ask why they did not report on X and if we can have their data or they want to collaborate<br>
</li>
<li>MT<br>
</li>
<li>MT MT MT<br>
</li>
<li>MT chr1<br>
</li>
<li>MT MT MT<br>
</li>
<li><pre><code>               Page 14 of 15  </code></pre></li>
<li></li>
<li>Ideas / notes - Sunday, 2 June 2024 at 15.21<br>
</li>
<li>- Plink chrX GWAS: https://zzz.bwh.harvard.edu/plink/faq.shtml#faq9<br>
</li>
<li>Genome-wide Association Studies (5cu) https://www.mv.helsinki.fi/home/mjxpirin/GWAS_course/<br>
</li>
<li>Does primate ILS in chr3 region track ILS of mtDNA?<br>
</li>
<li>Are any of the X/Y homologs associated with MT transport?<br>
</li>
<li>Can different mtDNA haplogroups of Bantu and West-African pygmys explain what looks like partial sweeps on chr3?<br>
</li>
<li>RHOT1 (AKA MIRO) is in the middle of the chr1 low-ILS region (along with<br>
</li>
<li>several others)<br>
</li>
<li>- RHOT1 has transcription factor binding sites in promoter for SRY (not expressed) and FOXO3 (peak in late round spermatids) and SRF (week but consistent peak in early round spermatids)<br>
</li>
<li>Mitochondrial dysfunction in autism spectrum disorders: a systematic review<br>
</li>
<li>and meta-analysis | Molecular Psychiatry <a href="https://www.nature.com/articles/mp2010136">https://www.nature.com/articles/mp2010136</a></li>
</ul>
</section>
<section id="xy-deg-msci-xi" class="level2">
<h2 class="anchored" data-anchor-id="xy-deg-msci-xi">XY-DEG-MSCI-XI</h2>
<ul>
<li>Do allele frequencies differ more between sexes on x than on autosomes<br>
</li>
<li>Xi only makes sense if its mechanism/machineery was allready in place before XY sex chromosomes. Otherwise females would have been better off maintaining dosage to control selections.<br>
</li>
<li>Maybe females maintain XI escape (and no dosage compensation) to remain in control of sex antagonism. See coop paper. See if XI escapers enriched for ASD in males are dosage compensated or not<br>
</li>
<li>Revisit idea about phase-seperation in “Meiotic sex chromosome inactivation and the XY body: a phaseseparation hypothesis”<br>
</li>
<li>Are deltaE genes more expressed because something that represses them “runs out”?<br>
</li>
<li>Do XI escapers or variable escapers have a characteristic expression trajectory?<br>
</li>
<li>Maybe the MSCI-XI connection is exposed by comparing species with different strata, different sex chromosome age of where one has an autosome fused to a sex chromosome like in slimy rats or in howler monkeys<br>
</li>
<li>Look at X minus Y expression in DE genes<br>
</li>
<li>- Compute deltaDE = (X - Y)/X expression across pseudo-time<br>
</li>
<li>- Make a plot of each gene with deltaDE across pseudo-time<br>
</li>
<li>- Make a k-means clustering and classify DE genes based on profiles<br>
</li>
<li>- Is the DE peaking at some particular times?<br>
</li>
<li>- Are nuclear DE genes different from cytoplasmic DE genes in that regard? - Are genes with a Y homolog different from the rest?<br>
</li>
<li>How are transcription factors (incl PRDM9) expressed? Do kmeans profiles for chrX TEs only<br>
</li>
<li>ILS<br>
</li>
<li>MT<br>
</li>
<li>MT chr3 MT MT chr3<br>
</li>
<li>MT<br>
</li>
<li><pre><code>        DEG MSCI XCI  </code></pre></li>
<li>DEG<br>
</li>
<li>Page 15 of 15<br>
</li>
<li></li>
<li></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../notes/hybrid-incomp.html" class="pagination-link" aria-label="Hybrid incomp">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-title">Hybrid incomp</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../notes/mbg-exercises.html" class="pagination-link" aria-label="MBG exercises">
        <span class="nav-page-text"><span class="chapter-title">MBG exercises</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>