<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Funding – Title of your project</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../notes/phasetype.html" rel="next">
<link href="../notes/meiotic-drive.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/notes.html"><span class="chapter-title">Funding</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">Title of your project</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Project</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Notes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/chromatin.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/funding.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/hybrid-incomp.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Hybrid incomp</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/ideas.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Ideas / notes</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/mbg-exercises.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">MBG exercises</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/meiotic-drive.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Meiotic drive</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/notes.html" class="sidebar-item-text sidebar-link active"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/phasetype.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Phase-type dist</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/prdm9.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">PRDM9</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/sex-antagonistic.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/spermatogenesis.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Spermatogenesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/strategy.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Research strategy</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/x-brain.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">X-brain</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/xi-msci.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Xi and MSCI</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul class="collapse">
  <li><a href="#questions-for-cliff-and-jeff" id="toc-questions-for-cliff-and-jeff" class="nav-link active" data-scroll-target="#questions-for-cliff-and-jeff"><strong>Questions for Cliff and Jeff:</strong></a></li>
  <li><a href="#correlation-between-y-x-and-a-trees" id="toc-correlation-between-y-x-and-a-trees" class="nav-link" data-scroll-target="#correlation-between-y-x-and-a-trees">Correlation between y x and a trees</a></li>
  <li><a href="#correlate-genes-in-different-ils-deserts-on-the-same-ancestral-branch" id="toc-correlate-genes-in-different-ils-deserts-on-the-same-ancestral-branch" class="nav-link" data-scroll-target="#correlate-genes-in-different-ils-deserts-on-the-same-ancestral-branch">correlate genes in different ILS deserts on the same ancestral branch</a></li>
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section"></a></li>
  <li><a href="#genomic-impact-of-species-hybridization" id="toc-genomic-impact-of-species-hybridization" class="nav-link" data-scroll-target="#genomic-impact-of-species-hybridization">Genomic impact of species hybridization</a>
  <ul class="collapse">
  <li><a href="#objectives-and-scientific-impact" id="toc-objectives-and-scientific-impact" class="nav-link" data-scroll-target="#objectives-and-scientific-impact">Objectives and scientific impact</a></li>
  <li><a href="#state-of-the-art" id="toc-state-of-the-art" class="nav-link" data-scroll-target="#state-of-the-art">State of the art</a></li>
  <li><a href="#research-plan" id="toc-research-plan" class="nav-link" data-scroll-target="#research-plan">Research plan</a></li>
  <li><a href="#feasibility-postdoc-training-schedule-and-publication-plan" id="toc-feasibility-postdoc-training-schedule-and-publication-plan" class="nav-link" data-scroll-target="#feasibility-postdoc-training-schedule-and-publication-plan">Feasibility, postdoc training, schedule and publication plan</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/notes.html"><span class="chapter-title">Funding</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-title">Funding</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p># FNU-1<br>
- Guy Sella approach with two related species with very different Ne.<br>
- Simulation with SLiM of dynamic drivers and invasion.<br>
- Project with Asger to fund Tobias?</p>
<p># FNU-2<br>
- Why is there no Denisova admixture on X? Is that related to X, Y or Mito?<br>
- Does admixture cease on on X before autosomes? Admixture of pairs with different divergence times.</p>
<section id="questions-for-cliff-and-jeff" class="level2">
<h2 class="anchored" data-anchor-id="questions-for-cliff-and-jeff"><strong>Questions for Cliff and Jeff:</strong></h2>
<ul>
<li>How many samples?: 1973: 400, later: 500<br>
</li>
<li>Is the sex known?: yes, even with geolocations, known social groups and basic body metrics<br>
</li>
<li>In what shape? Ok for Illumina<br>
</li>
<li>How far into both “pure populations”?<br>
</li>
<li>How smoothly is there sampled along the admixture gradient?<br>
</li>
<li></li>
</ul>
<p>Apply for 50 genomes 950</p>
<p>anubis (weetern etiohopa) - hamadryas (filoha 20)</p>
<ul>
<li></li>
</ul>
<p>Tory bergmen gelada<br>
Fred brett papeer</p>
<p>Hybrid index score phenotypic 1 - 3, rings on hair, maybe use that for gene mapping…</p>
<p>Assets:</p>
<ul>
<li>Expertise<br>
</li>
<li>Precious unique samples, and many, found collaborators, a unique opportunity<br>
</li>
<li>Pilot data not from a hybrid zone, showing both positive and negative selection (better data and targeted analysis will make it look even better)</li>
</ul>
<p>State of art and motivation:</p>
<ul>
<li>Drivers of speciation and impact of hybridization of existing species.<br>
</li>
<li>Impact on the evolution of species in contact<br>
</li>
<li>BDMIs have been looked at in fish, but not in a species close to humans. Only done in swordtail fish - I want to do it in a close relative to humans…<br>
</li>
<li>A long history of hybridization in baboons. I collaborate with the largest authority on baboon biology.<br>
</li>
<li>Quantify epistatic interactions.<br>
</li>
<li>Meiotic drive (as one hypothesis)<br>
</li>
<li>Unique data set.<br>
</li>
<li>Include Tung data set (did not look at epistasis or X)<br>
</li>
<li>Eriks plot of chr8 and chrX<br>
</li>
<li>My own work/contributions to this in great apes, my human sweeps on X<br>
</li>
<li>Use close relative to humans to use human gene and genome annotation and knowledge of expression gene and of the interaction of genes.<br>
</li>
<li>Learn about baboons, learn what we cannot about human archaic admixture (gone and had small Ne), but most importantly learn general mechanisms<br>
</li>
<li>The autosomal, X, Y, Mitochondrial introgression examples from Neanderthals and Denisovans<br>
</li>
<li>Begin by explaining what can be done with autosomes (with most data) and then say that the X is probably where most of it happens<br>
</li>
<li>Haldane’s rule (have some old text on that).</li>
</ul>
<p>Perspectives:</p>
<ul>
<li>Drivers of speciation, which genes or genomic regions?<br>
</li>
<li>Are BDMIs mainly X/Y or X/A incompatibilities?<br>
</li>
<li>Why do species that hybridize a lot remain genetically, phenotypically, and behaviorally distinct?<br>
</li>
<li>Knowledge of epistatic interactions can aid our understanding of gene-gene interactions and networks.<br>
</li>
<li>Improve our understanding of how much genetic variation that is due to epistasis rather than additive.</li>
</ul>
<p>Questions:</p>
<p>Autosomes:</p>
<ul>
<li>What is the “degree” of epistatic interactions? How many othe genes need to be with same ancestry as the gene itself? One? Two? Ect. This “degree” should influence the dependency on minor parent ancestry: The more genes a gene depends on, the more with the minor ancestry proportion matter. Extreme cases: no minor ancestry proportion is small enough if a gene depends on all other genes. If a gene depends only on the ancestry of a single other gene, the negative selection will fall dramatically with minor ancestry proportion.<br>
</li>
<li>Does the correlation with recombination go away in the middle of the hybridization zone?<br>
</li>
<li>How quickly does it change?<br>
</li>
<li>What can we learn about epistatic interactions from the dependency on minor proportion?<br>
</li>
<li>How much is explained by local Ne, how much by recombination and how much by minor parent proportion?<br>
</li>
<li>Controlling for recombination rate, what is the strength of negative selection (BDMI) as function of minor parent ancestry proportion?<br>
</li>
<li>What can we learn by use segment length as a proxy for age since introgression? Longer segments have had not had a long time to be purged by selection and may not be informative of selection (on the other hand they likely have more deleterious mutations because of their length). What happens to the introgression landscape if we stratify by segment length.<br>
</li>
<li>Purging speed (strength of negative selection) as a function of minor parent ancestry proportion (MPAP): Can we measure that empirically and compare to a theoretical prediction parameterized by amount of epistatic interaction? <strong>Ask Thomas for advice</strong><br>
</li>
<li>How prevalent is adaptive introgression?<br>
</li>
<li>Can we estimate the expected density of age given the segment map-length?<br>
</li>
<li>Can we compare that neutral expectation with the observed (inferred) age of segments of different length using Relate?<br>
</li>
<li>Can we do AD-scans?<br>
</li>
<li>Comparing hybridization of Anubis to both hamadryas and yellows. With the different divergences.<br>
</li>
<li>Forward simulations like Harris and Nielsen did with Neanderthals to identify scenarios able to explain observations.<br>
</li>
<li></li>
</ul>
<p>X:</p>
<ul>
<li>Can we explain Haldane’s Rule?<br>
</li>
<li>Does correlation with recombination change more abruptly on X?<br>
</li>
<li>Is negative selection stronger on X?<br>
</li>
<li>Are there desserts of isolation on X?<br>
</li>
<li>Do desserts sometimes produce introgressing sweeps?<br>
</li>
<li>Do “pure ancestry” regions on X arise from lack of ILS (recurrent sweeps in ancestors) and/or from negative selection against introgression?<br>
</li>
<li>Estimate demography separately from autosomes and see if X/A pi can be explained via the Pool-Nielsen effect.</li>
</ul>
<p>Method:</p>
<ul>
<li>Tweak Clues to infer adaptive introgression<br>
</li>
<li>Use X/A contrast to distinguish selection, demography and sex-biased admixture.<br>
</li>
<li>Use Relate trees to infer admixture.<br>
</li>
<li>AD scans using Admixture</li>
</ul>
<p>Tasks:</p>
<ul>
<li>Sequence and call variation, phasing<br>
</li>
<li>Create recombination maps<br>
</li>
<li>Use Relate trees to infer ancestry segments and estimation of ancestry proportions<br>
</li>
<li>Analysis of negative selection and epistasis across hybrid zone (MPAP ~ rec rate, MPAP1 ~ MPAP2 )<br>
</li>
<li>Development of models to quantify epistatic interactions. Validate models by forward simulations.<br>
</li>
<li>AD scans for candidate BDMI genes or pathways.<br>
</li>
<li>Scan for sweeps using Clues (adaptive introgression of BDMI genes on X and auto?)<br>
</li>
<li>Correlations to body measures and hybrid index for each individual<br>
</li>
<li></li>
</ul>
<p>Budget:</p>
<ul>
<li>Already have ~400,000 DKK -&gt; 100 genomes<br>
</li>
<li>Ask for another half 400,000 for sequencing</li>
</ul>
<p><em>Finally, X and Y chromosomes also show different introgression patterns from autosomes in meetings between anatomically modern humans and Neanderthals <a href="https://app.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b/all?uuid=9868662773544371&amp;item_ids=ff394a98-20f7-4882-871c-840e9f73d65b:04cf608e-622a-4bc4-a011-a2fe844ea6a3,ff394a98-20f7-4882-871c-840e9f73d65b:f61b0ae5-23d7-45e7-b4f0-b644ce25006b,ff394a98-20f7-4882-871c-840e9f73d65b:ace6f4b6-b49c-4a40-b4ba-35916ab423eb">(Petr et al.&nbsp;n.d.; Sankararaman et al.&nbsp;2016; Schierup 2020)</a>. In an early meeting more than 250,000 years ago, the Neanderthal Y chromosome was replaced by the modern human version <a href="https://app.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b/all?uuid=22775545431605082&amp;item_ids=ff394a98-20f7-4882-871c-840e9f73d65b:f61b0ae5-23d7-45e7-b4f0-b644ce25006b">(Petr et al.&nbsp;n.d.)</a>, and the X chromosome introgressed at least as much as the autosomes <a href="https://app.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b/all?uuid=7031522705435261&amp;item_ids=ff394a98-20f7-4882-871c-840e9f73d65b:cf52eebe-bdb9-4a55-ba62-bd24a8ad16f5">(Hubisz et al.&nbsp;2020)</a>. In the later meeting around 55,000 years ago, the direction of admixture was reversed. Here the Neanderthal Y chromosome did not introgress into the out-of-Africa anatomically modern humans and the X chromosome introgressed to a much smaller extent (0.3%) than the autosomes (1.4) <a href="https://app.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b/all?uuid=6031336742837665&amp;item_ids=ff394a98-20f7-4882-871c-840e9f73d65b:04cf608e-622a-4bc4-a011-a2fe844ea6a3">(Sankararaman et al.&nbsp;2016)</a>. Large regions of the X chromosome today are devoid of Neanderthal introgression - coined as Neanderthal deserts - and overlap with the regions of the X chromosome repeatedly targeted by natural selection in great apes, as discussed above.</em></p>
</section>
<section id="correlation-between-y-x-and-a-trees" class="level2">
<h2 class="anchored" data-anchor-id="correlation-between-y-x-and-a-trees">Correlation between y x and a trees</h2>
</section>
<section id="correlate-genes-in-different-ils-deserts-on-the-same-ancestral-branch" class="level2">
<h2 class="anchored" data-anchor-id="correlate-genes-in-different-ils-deserts-on-the-same-ancestral-branch">correlate genes in different ILS deserts on the same ancestral branch</h2>
</section>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section"></h2>
</section>
<section id="genomic-impact-of-species-hybridization" class="level1">
<h1>Genomic impact of species hybridization</h1>
<p>Divergence of populations to the creation of new species and their<br>
subsequent contact and hybridization is ubiquitous in evolution. Hybridization<br>
Dispite its fundamental importance little is known about the genes that convers incompatibility of hybrids, weather many or few, and which ones and their function. Taking our understanding of the genomic impact of hybridization (or barriers to introgression) to a new level will require analysis of a densely sampled hybridization in a species closely so closely to humans that the ample annotation from humans can be used.</p>
<p><em>Genetic recombination is an essential and mysterious process at the heart of evolution. It shuffles</em><br>
<em>genetic variants to allow organisms to adapt to new environments and it promotes natural selection</em><br>
<em>by exposing individual fitness effects of genetic variants. Despite its immense importance, little is</em><br>
<em>known about the mechanisms that control recombination and how these mechanisms evolve. Taking</em><br>
<em>our understanding of recombination to a new level will require an approach that is able to measure</em><br>
<em>the long-term evolution of recombination in individual species and then link this long-term change</em><br>
<em>in recombination to the responsible genomic changes.</em></p>
<section id="objectives-and-scientific-impact" class="level2">
<h2 class="anchored" data-anchor-id="objectives-and-scientific-impact">Objectives and scientific impact</h2>
</section>
<section id="state-of-the-art" class="level2">
<h2 class="anchored" data-anchor-id="state-of-the-art">State of the art</h2>
<p>pilot</p>
</section>
<section id="research-plan" class="level2">
<h2 class="anchored" data-anchor-id="research-plan">Research plan</h2>
<p>intro</p>
<section id="task-1" class="level3">
<h3 class="anchored" data-anchor-id="task-1">Task 1:</h3>
</section>
<section id="task-2" class="level3">
<h3 class="anchored" data-anchor-id="task-2">Task 2:</h3>
</section>
<section id="task-3" class="level3">
<h3 class="anchored" data-anchor-id="task-3">Task 3:</h3>
</section>
<section id="task-4" class="level3">
<h3 class="anchored" data-anchor-id="task-4">Task 4:</h3>
</section>
<section id="task-5" class="level3">
<h3 class="anchored" data-anchor-id="task-5">Task 5:</h3>
</section>
</section>
<section id="feasibility-postdoc-training-schedule-and-publication-plan" class="level2">
<h2 class="anchored" data-anchor-id="feasibility-postdoc-training-schedule-and-publication-plan">Feasibility, postdoc training, schedule and publication plan</h2>
<section id="state-of-the-art-1" class="level3">
<h3 class="anchored" data-anchor-id="state-of-the-art-1">State of the art</h3>
<p>In all but a very few cases, we do not know which genes that are responsible for the reduced fertility of hybrids that eventually lead to formation of new species. We do know, however, that when incompatibility does arise, it almost always first appears as reduced fertility of the heterogametic sex (E. g XY mammal males or ZW female birds). This phenomenon has been dubbed Haldane’s rule and represents a century of empirical observations. In the past few years, genomic analyses of individual species groups have confirmed that portions of X and Z chromosomes resist genetic exchange among closely related species. Still, the responsible genes remain unknown, and the evolutionary mechanisms by which incompatibilities arise remain poorly understood. The Dobzhansky-Muller model proposes that incompatibility arises when sets of coevolving inter-dependent genes in each population diverge to a point where combinations of genes from different populations no longer function.<br>
I and others have proposed that a rapid buildup of incompatibilities between sex chromosomes is fueled by an arms race between competing selfish genes on each chromosome (Figure 1A). Only small parts of the sex chromosomes recombine in meiosis. A consequence of this is that genes on sex chromosomes can promote their own transmission to the next generation. E.g. an X-linked gene can control its own selection by any means that leave more viable sperm cells carrying X chromosomes than carrying Y chromosomes. It can do so either by influencing the symmetry of meiosis or by killing sperm cells that carry the Y chromosome (Meiklejohn and Tao 2010). Such meiotic drive will promote the selfish gene variants, even if these are harmful to the organism’s fitness and distort the sex-ratio of offspring. Sex chromosome meiotic drive has been observed in several species (Jaenike 1999), and its genetic basis has recently been proposed in both fruit files (Bachtrog, Ellison et al.&nbsp;2018) and mice (Rathje, Johnson et al.&nbsp;2019). In these cases, the responsible genes are ampliconic, and such genes are also found in primates. However, any sex-linked critical gene expressed during or after meiosis has the potential to act as a meiotic driver. I have previously shown that, unlike the remaining genome, the X chromosome was targeted by strong recurrent selection across great ape evolution (Nam, Munch et al.&nbsp;2015, Munch, Nam et al.&nbsp;2016). Most recently, I reported that the X chromosomes of non-African humans were hit by bursts of very strong selection between 45 and 55 thousand years ago (Skov, Macia et al.&nbsp;2018). Together, these observations strongly support the action of meiotic drive.<br>
An arms-race between meiotic drivers on X and Y (or Z and W) will accelerate the coevolution of these genes. This will lead to faster divergence of sex chromosomes between populations and eventually induce reproductive incompatibilities between X and Y chromosomes from different populations (Figure 1). This arms-race has been observed in mice (Rathje, Johnson et al.&nbsp;2019) and has the potential to explain the widely observed infertility of hybrid XY males and ZW females (Coyne and Orr 2004). It may also explain the observation that genes on the X chromosome contribute disproportionately to sterility (Masly and Presgraves 2007).<br>
I propose that meiotic balance is maintained by antagonizing meiotic drivers on separate sex chromosomes that depress and stabilize each other’s frequency. Perturbations of this equilibrium will trigger strong temporary selection on individual meiotic drivers. In ongoing work, I use forward simulations to show that competing meiotic drivers can be kept at low frequency by frequency-dependent selection and that the resulting dynamic equilibrium maintains meiotic balance (Figure 2). I also find that admixture or demographic perturbations disturb this equilibrium, triggering strong temporary selection on individual driver variants. The hypothesis can thus explain my observations of strong selection on the X chromosome coinciding with the exodus of modern humans from Africa (Skov, Macià et al.&nbsp;2018). The hypothesis also reconciles strong selection on individual meiotic drivers with empirical observations of stable and even sex ratios in most species.<br>
Because meiotic drive reduces fertility and distorts the sex-ratio, I propose that these detrimental effects trigger compensating adaptive selection on normal genes across the genome to restore fertility and an even sex ratio. Such compensation may explain why genes in tumor suppression pathways are among the most rapidly adapting genes in the genome (Nielsen, Bustamante et al.&nbsp;2005). This adaptation is unexpected because these genes perform cellular tasks that serve core cellular functions, which are not expected to require much if any, adaptive evolution. Such genes are found in the TP53 signaling pathway that controls DNA damage response and apoptosis. This regulatory network strongly overlaps networks of genes controlling recombination and meiosis. Many of these genes are located on the X chromosome, and my recent report (Skov, Macià et al.&nbsp;2018) suggests that several of them may be meiotic drivers (E.g., HUWE1, and SMC1A). I hypothesize that the selection of such X-linked drivers may perturb the overlapping networks to induce apoptosis of Y carrying sperm, thus producing a preferential transmission of X carrying sperm. This detrimental effect on male fertility is then expected to trigger compensating adaptive evolution of other genes in the same networks to counter this regulatory perturbation and restore male fertility and an even sex ratio of children. In a pilot study addressing this hypothesis, I found that the 3p21.3 genomic region has been affected by recurrent adaptation, and it indeed harbors an abundance of genes in the TP53 network: IP6K2, USP4, HYAL2, HYAL3, and PRKAR2A.<br>
I thus present a general model of the genetic basis of speciation and its derived effects across the genome. The model not only explains existing empirical evidence but also has the potential to guide new insights into speciation and fertility.</p>
</section>
<section id="objectives-and-scientific-impact-1" class="level3">
<h3 class="anchored" data-anchor-id="objectives-and-scientific-impact-1">Objectives and scientific impact</h3>
<p>Although the genetic basis of hybrid incompatibility will vary across species groups, each one is an individual outcome of a similar evolutionary process. By accumulating evidence across species groups, I will thus be able to learn which classes of genes and regulatory networks that are responsible, and to identify general mechanisms by which mutation, selection, and recombination contribute to the accumulation of incompatibility. I will also be able to detect differences across groups of species, attributed to the sex-determination system (XY vs.&nbsp;ZW), population size, dispersal range, and sperm competition. In each species, I will identify candidate meiotic drivers by locating strong selection on X and Z chromosomes. I will link each candidate driver gene to hybrid incompatibility by testing if their exchange between species is associated with reduced fitness.<br>
This project is the first to link natural selection in the form of meiotic drive to hybrid incompatibility across many groups of species. By answering fundamental evolutionary questions, the results from this project will impact many connected research fields. Being able to evaluate from genomic data alone, which populations are emerging new species, will inform biodiversity management. Understanding how tumor-suppression adapt to counter regulatory disruptions by meiotic drivers will provide new insights into the regulatory dependencies of tumor-suppressors. New insights into the sensitivities of male fertility will identify new genes and pathways as potential targets for explaining and treating the subfertility of one-third of Danish men.</p>
</section>
<section id="data" class="level3">
<h3 class="anchored" data-anchor-id="data">Data</h3>
<p>To maximize inference power, I will analyze all available full genome data sets from species groups with different characteristics, such as XY or ZW karyotype (E. g. birds or bats), dispersal range (E. g. birds vs monkeys), population size (E. g. mosquitos vs polar bears), and divergence time (E. g. ape species vs ape populations). Analysis workflows will be designed to allow the seamless addition of the many additional genomic data sets that will become available during the project period. I now have access to the following full genome data, either through public access, or early access through collaborations with (Guojie Zhang DK, Jeffrey Rodgers, US, and Tomas Marques-Bonet, Spain): &gt;1000 humans across 51 populations, 43 Gorillas across three subspecies, 26 orangutans across two species, 58 chimpanzees across four subspecies, 13 bonobos, 105 baboons across five subspecies, 163 vervet monkeys across five populations, 89 bears across two species, 60 bats across three species, 56 finches across four species, 60 penguins across three species, 690 swordtail fish across four species, 1142 mosquitos across two species and six populations. With a few exceptions, each data set includes both pure unadmixed and hybrid populations/species. In addition, I have access to a single genome of a large number of species: 55 primates, 344 birds, 854 butterflies, and 81 flatworms and roundworms. Where genome samples from admixture zones are not available, I will acquire such samples through collaborations and perform supplementary genome sequencing myself. The project budget covers sequencing of 300 such samples.</p>
</section>
<section id="automation-of-reproducible-hpc-workflows" class="level3">
<h3 class="anchored" data-anchor-id="automation-of-reproducible-hpc-workflows">Automation of reproducible HPC workflows</h3>
<p>Goal: establish automated HPC workflows. The key to the success of this project is the scale of parallel analyzes so that each of the computationally demanding analyses described below can be applied to any number of large data sets on our HPC facility (GenomeDK). This requires a high level of automation that I will achieve by embedding all analyses in one pipeline. To this end, I will use a workflow library (GWF) that we have developed at the Bioinformatics Research Centre (BiRC). This tool graphs dependencies between analyses to maintain the integrity of the pipeline as analyses update and data sets are added. All steps from the initial curation of raw data to the final summary analyses as Jupyter notebooks will be embedded in this workflow. I will use Conda (by Anaconda) to maintain reproducible library dependencies, and all developed analyses will be distributed as self-contained Conda packages or Docker images. This allows collaborating research groups, with access to proprietary or sensitive data, to run the analyses in-house and only contribute anonymized meta-data to this project.</p>
</section>
<section id="meiotic-drivers-in-living-species" class="level3">
<h3 class="anchored" data-anchor-id="meiotic-drivers-in-living-species">Meiotic drivers in living species</h3>
<p><strong>Goal: develop tailored methods for inference of sweeps.</strong> Recent adaptive evolution manifests as selective sweeps of genetic variation. However, meiotic drive may not be recent and is expected to produce incomplete sweeps, which leave a similar, but more subtle, genetic signature. I will develop new specialized methods able to identify this particular signal.<br>
I will develop a suite of population genetic summary statistics, each tailored to the particular inference of candidate regions subject to non-recent incomplete sweeps. This will be done in collaboration with David Enard (University of Arizona), who has already developed one such statistic, hapDAF, which quantifies linkage of high-frequency variants. This statistic has very good power to detect simulated non-recent partial sweeps. Tailored summary statistics will initially be combined into a composite score using a framework following (Grossman, Shlyakhter et al.&nbsp;2010). I will then develop a more powerful approach by training deep convolutional neural networks on vectors of summary statistics to classify each genomic region as subjected to a sweep or not.<br>
For a detailed analysis of causal variants in swept regions, I will develop a tailored method, which extends a recently published model (Stern, Wilton et al.&nbsp;2019). This method integrates over marginal genealogies at a focal site to test if inferred frequency trajectories of candidate alleles are unlikely without selection. I will modify the likelihood model to describe a confined non-recent episode of strong selection. By incorporating demography in the null model, this approach is also able to distinguish the similar effects of population bottlenecks and selective sweeps. In addition to allowing a more powerful estimation of selection strength, this method will allow me to test my hypothesis that strong selection on meiotic drivers is episodic and does not lead to the fixation of drivers. This test is possible because the developed approach will enable me to estimate the historical allele frequency of meiotic drivers.<br>
I will apply both methods to all population genomic diversity data for all available species and populations to identify candidate meiotic drivers on the X and Z sex chromosomes. To address the hypothesis that demographic events may trigger strong temporary selection on drivers, I will estimate the historical population size of each species, using HMM-based methods (SMC++, and MSMC). I will then relate the strength and number of identified sweeps to the occurrence of a demographic event such as population bottlenecks.</p>
</section>
<section id="meiotic-drivers-in-ancestral-species" class="level3">
<h3 class="anchored" data-anchor-id="meiotic-drivers-in-ancestral-species">Meiotic drivers in ancestral species</h3>
<p><strong>Goal: find sweeps in ancestral species.</strong> Whereas patterns of genetic diversity reveal the recent action of selection, I need to analyze genomic patterns of divergence between multiple species to reveal the effect of the selection across long evolutionary time scales. The common ancestor of two sequences sampled from different species is much older than the time they stopped interbreeding. If the population size of the ancestral species is large, the common sequence ancestor may reach all the way back into an additional species shared by a third species. In this case, the topology of individual gene trees may differ from the topology of the species tree. This is a phenomenon called incomplete lineage sorting (ILS), which I have worked on for many years (Figure 3).<br>
For all relevant trios of closely related species, I will use a coalescent hidden Markov model (Dutheil, et al.&nbsp;2009), to infer the gene tree topology for each consecutive segment of the aligned genomes using both Viterbi and posterior decoding of each trained HMM. The vast amount of data produced will be stored efficiently as tree-sequence data structures using tsKit. Strong selective sweeps, produced by meiotic drivers in ancestral species, will induce immediate ancestry of lineages. Completed hard selective sweeps are thus readily detectable as wide genomic regions without any ILS (Figure 3C). Soft or partial sweeps are visible as wide genomic regions with a homogeneously reduced, but non-zero, proportion of ILS. I can thus test my hypothesis that meiotic drivers sweep when the equilibrium of antagonizing meiotic drivers is disturbed. In this case, soft sweeps (selection on pre-existing variants) and non-recent partial (aborted) sweeps should be much more prevalent on the X and Z chromosomes than on the remaining genome. I will evaluate the significance of each low-ILS region by comparing its size, distribution of segment lengths, and autocorrelation of ILS proportion, to empirical background distributions obtained from the entire genome (accounting for the difference in effective population sizes of sex chromosomes and autosomes).<br>
I will identify sweeps on all chromosomes of each entire ancestral genome. However, in line with my previous work (Dutheil, Munch et al.&nbsp;2015, Nam, Munch et al.&nbsp;2015, Munch, Nam et al.&nbsp;2016), I expect to find most of them on the X and Z chromosomes. Indeed, in a preliminary unpublished analysis of ILS on the X chromosome of the human-chimpanzee and human-orangutan ancestors I find several strong sweeps in these ancestors. I will detect individual sweeps in all ancestral species, but also combine gene tree information for all species trios, to produce gene trees spanning the entire 55 primates, 344 birds, 854 butterflies, and 81 worms. This way I will combine evidence of selection across ancestral species.</p>
</section>
<section id="the-time-scale-of-reproductive-isolation" class="level3">
<h3 class="anchored" data-anchor-id="the-time-scale-of-reproductive-isolation">The time scale of reproductive isolation</h3>
<p><strong>Goal: learn when admixture occurred between diverging populations.</strong> I will develop and apply a new method to infer historical admixture between species and populations. Knowledge of such admixture is required to test if admixture ceases for sex-chromosomes before it does for the remaining genome. It is also necessary to test the hypothesis that events of secondary contact between diverging population trigger strong meiotic drive, as suggested by my recent report on the human expansion from Africa (Skov, Macià et al.&nbsp;2018).<br>
Recent admixture between species or populations will be inferred using published software (see below) and dated from the length distribution of genomic admixture segments. State of the art for estimating older admixture exploits the covariance structure of genetic diversity between populations. These methods, however, perform a greedy search for an admixture graph with a good fit to data and do not provide an estimate of confidence in the obtained result. I propose a new approach to inference that does not have these shortcomings. Recent work has shown that the coalescent, which models the ancestry of sampled genes, can elegantly be described using phase-type distributions (Hobolth, Siri-Jegousse et al.&nbsp;2019). In this new formulation, it is possible to obtain the full (multivariate) distribution of the frequency spectrum of variants (site frequency spectrum) in a single population. Together with a MSc student, I have shown that the phase-type formulation maps directly to directed acyclic graphs for the relevant models. This insight dramatically reduces the running time, allowing for up to 50 samples and thus powerful inference. Using this approach, I have also shown a general expression for the joint distribution of branch lengths of gene tree lineages with different numbers of descendants. Preliminary results show that I will be able to compute the joint likelihood of the site frequency spectra from two populations with common ancestry. I will extend this simple setting to three populations with the possibility of admixture between species. Because we can compute the likelihood of an observed multi-population SFS given a species-admixture graph, we can use Markov chain Monte Carlo (MCMC) to obtain posterior probabilities of such graphs as well as the marginal probabilities of individual admixture events. Having obtained an admixture graph, I will estimate the dates of species splits and admixture events using a coalescent HMM.</p>
</section>
<section id="the-genomic-location-of-reproductive-barriers" class="level3">
<h3 class="anchored" data-anchor-id="the-genomic-location-of-reproductive-barriers">The genomic location of reproductive barriers</h3>
<p><strong>Goal: learn if candidate meiotic drivers are the first genes that induce hybrid incompatibility.</strong><br>
Having identified candidate meiotic drivers, I will investigate if their rapid evolution induces a lower fitness of hybrids. In this case, the candidate genes will resist admixture and produce genomic regions around them that are free from genomic segments introduced from other populations or subspecies – so-called “admixture deserts”. There is ample empirical evidence that both X and Z chromosomes harbor an abundance of such wide regions. However, the regions contain a large number of potentially responsible genes. Because the size of the admixture desert on each side of a causal gene is the outcome of a stochastic process, I will be able to sharply reduce the number of gene candidates by limiting the search to admixture deserts shared by many related species. Prediction of admixed segments along the genome will be made using published methods. To detect recent admixture between sampled populations, I will use the HMM-based ChromoPainter and HMM+MCMC based RELATE and ARGweaver-D to identify admixed genomic segments. To detect admixture segments contributed from unsampled populations, I will use a method using conditional random fields. I will also use the HMM-based method, which I used to detect archaic introgression into modern humans in my recent report of strong selection on the human X chromosome.</p>
</section>
<section id="invasion-of-meiotic-drivers" class="level3">
<h3 class="anchored" data-anchor-id="invasion-of-meiotic-drivers">Invasion of meiotic drivers</h3>
<p><strong>Goal: identify meiotic drivers that invaded other populations.</strong> Perhaps counterintuitively, the meiotic drivers, which fuel accelerated coevolution of sex-linked genes and the buildup of incompatibility between populations, can themselves invade other subspecies (Meiklejohn, Landeen et al.&nbsp;2018). This will happen if they are able to act as meiotic drivers in the recipient subspecies. Such introgressions constitute a separate type of observations, which can be used to identify meiotic drivers. To infer such introgressions, I will develop a new approach based on a newly published study (VolcanoFinder), which models a characteristic but weak signature in genetic diversity. In the particular case of non-African humans, the statistical power can be increased ten-fold by analyzing only genetic variation unique to non-Africans. In the many other species, the same trick can be used for a similar performance boost. I will also develop an approach based on called genomic admixture segments that quantify if the sharing of segments is more prevalent than expected from population structure. These methods will be applied to all available admixing species. Historical invasions between ancestral species will be identified from ILS patterns as extended regions showing a gene tree different from the species tree. Here, I will call individual ancestral invasions, but also to identify the accumulated signal of repeated invasions of a gene across a species tree from a significantly elevated amount of ILS.</p>
</section>
<section id="compensating-autosomal-adaptation" class="level3">
<h3 class="anchored" data-anchor-id="compensating-autosomal-adaptation">Compensating autosomal adaptation</h3>
<p><strong>Goal: learn how sex-linked drive affects adaptation across the genome.</strong> Many of the most rapidly adapting genes in the genome are part of sizeable regulatory gene networks linked to tumor suppression. I will expand the inference of selective sweeps in each species to include the entire genome. This will allow me to investigate if the adapting genes are in the same pathways as meiotic drivers and thus build support for my hypothesis that meiotic drive triggers compensating adaptation of these autosomal genes. This will be done using published software for signaling pathway impact analysis (SPIA).<br>
In a complementary approach, I will test if the same is true for adaptive evolution over long evolutionary time scales. I will compare genetic polymorphism and divergence at functional and nonfunctional sites to estimate the rate of adaptive evolution of each gene in the genome. To this end, I will use a method, which I recently applied to estimate the amount of adaptive evolution driven by virus interaction (Castellano, Uricchio et al.&nbsp;2019). I will then subject the network of known protein interactions from the KEGG, REACTOME, and SPRING gene network databases to a network propagation algorithm. This approach will allow me to test if recurrently adapting autosomal genes are more strongly connected to candidate meiotic drivers than expected (when properly accounting for network architecture and tissue expression).</p>
</section>
<section id="meta-analyses" class="level3">
<h3 class="anchored" data-anchor-id="meta-analyses">Meta-analyses</h3>
<p><strong>Goal: accumulate and contrast evidence across species to produce new insight.</strong> In contrast to the complex computational analyzes of each species, the results produced are simple. The inference of selective sweeps will produce genomic regions containing a small number of candidate genes. Inference of hybrid incompatibility will produce genomic regions where admixture between populations is precluded by reduced fitness (admixture deserts). Although each genomic region is associated with a measure of significance and analysis meta-information, the size of meta datasets will be small enough to handle in Jupyter notebooks hosted from a powerful HPC compute node. This allows for manual inspection, visualization, and data analysis of the continually accumulating results.<br>
I will use a published statistical method for cross-species gene set analysis (XGSA) to identify pathways and regulatory networks with an overrepresentation of candidate genes. By testing all candidate genes for enrichment of ontologies terms, I will further establish which cellular functions are primarily responsible for hybrid incompatibility. I will combine these analyses with inferred species divergence, admixture, and demography, as well as the ecological, physical, and behavioral properties of each species. That way, I will learn if the responsible genes, pathways, or characteristic cellular functions differ in species with XY and ZW sex-determination system, or if they differ at early and late stages of speciation. I will also learn if meiotic drive is more frequent, and speciation rate faster, in species with no sperm competition. This is expected because detrimental effects on fertility have a smaller effect on fitness in such species.</p>
</section>
<section id="project-feasibility" class="level3">
<h3 class="anchored" data-anchor-id="project-feasibility">Project feasibility</h3>
<p>The progress and completion of this project rely on access to an HPC facility, which is available through the Bioinformatics Research Centre (GenomeDK). All data is either publicly available or available through collaborators. The feasibility of the proposed methods has been ensured through pilot projects of my MSc and project students. It is possible that one or more of my hypotheses are not proven true. However, the bulk of empirical evidence leaves it unquestionable that sex-chromosomes hold the key to understanding speciation. Hence even in the unlikely case that my hypotheses are wrong, the scale and breadth of this project will reveal the true mechanisms. In either case, the general interest in these questions will ensure publications of high impact.</p>
</section>
<section id="work-and-publication-plan" class="level3">
<h3 class="anchored" data-anchor-id="work-and-publication-plan">Work and publication plan</h3>
<p>The research project constitutes the work of three postdocs and two Ph.D.&nbsp;students. One two-year postdoc will work on new methods for the inference of selective sweeps from diversity data. One two-year postdoc will work on autosomal adaptation in response to meiotic drive. One three-year postdoc will work on hybrid incompatibility and invasion of meiotic drivers. One Ph.D.&nbsp;student will work on the inference of selection and admixture using incomplete lineage sorting. One Ph.D.&nbsp;student will work on phase-type distributions and admixture graphs. All students and postdocs will take part in workflow integration and meta-analyses. See Figure 4 for an activity plan. Students and postdocs will lead publications of their own methods or analysis work and coauthor papers on results from integrating all analyses into the broader workflow.</p>
</section>
<section id="recruitment-and-training" class="level3">
<h3 class="anchored" data-anchor-id="recruitment-and-training">Recruitment and training</h3>
<p>I have already identified an ideal candidate for the Ph.D.&nbsp;project dedicated to working on phase-type distributions. Tobias Røikjer has written his master thesis on this topic and has been instrumental in executing the pilot project that forms the basis of the method proposed. I will recruit postdocs and an additional Ph.D.&nbsp;student internationally, using my network of past and present collaborators, particularly at Uppsala University, UC Berkeley, Harvard, Max Planck in Leipzig, and Pompeu Fabra University in Barcelona. Ph.D.&nbsp;students will be enrolled in the Graduate School of Science (GSST) and Technology at Aarhus University. Postdocs and Ph.D.&nbsp;students will be part of a study environment with international postdocs and Ph.D.&nbsp;students from other groups, all working on evolutionary questions using similar theory and methodology. In addition to the mandatory activities associated with GSST, Ph.D.&nbsp;students at our research center are fully integrated into all academic activities such as seminars and journal clubs. Postdocs will be encouraged to participate in my courses in population genomics and data science. Project students and MSc students will be trained in the context of this project and be assigned small self-contained subprojects addressing scientific questions embedded in the larger research strategy.</p>
</section>
<section id="references" class="level3">
<h3 class="anchored" data-anchor-id="references">References</h3>
<p>Bachtrog, D., C. Ellison, C. Leonard, E. Landeen, L. Gibilisco and N. Phadnis (2018). “Rampant cryptic sex chromosome drive in Drosophila.” bioRxiv.<br>
Castellano, D., L. H. Uricchio, K. Munch and D. Enard (2019). “Viruses rule over adaptation in conserved human proteins.” bioRxiv: 555060.<br>
Coyne, J. A. and H. A. Orr (2004). Speciation, Sinauer Associates.<br>
Dutheil, J. Y., K. Munch, K. Nam, T. Mailund and M. H. Schierup (2015). “Strong Selective Sweeps on the X Chromosome in the Human-Chimpanzee Ancestor Explain Its Low Divergence.” PLoS Genet 11(8): e1005451.<br>
Grossman, S. R., I. Shlyakhter, E. K. Karlsson, E. H. Byrne, S. Morales, G. Frieden, E. Hostetter, E. Angelino, M. Garber, O. Zuk, E. S. Lander, S. F. Schaffner and P. C. Sabeti (2010). “A composite of multiple signals distinguishes causal variants in regions of positive selection.” Science 327(5967): 883-886.<br>
Hobolth, A., A. Siri-Jegousse and M. Bladt (2019). “Phase-type distributions in population genetics.” Theor Popul Biol 127: 16-32.<br>
Jaenike, J. (1999). “Suppression of Sex-Ratio Meiotic Drive and the Maintenance of Y-Chromosome Polymorphism in Drosophila.” Evolution 53(1): 164-174.<br>
Masly, J. P. and D. C. Presgraves (2007). “High-resolution genome-wide dissection of the two rules of speciation in Drosophila.” PLoS Biol 5(9): e243.<br>
Meiklejohn, C. D., E. L. Landeen, K. E. Gordon, T. Rzatkiewicz, S. B. Kingan, A. J. Geneva, J. P. Vedanayagam, C. A. Muirhead, D. Garrigan, D. L. Stern and D. C. Presgraves (2018). “Gene flow mediates the role of sex chromosome meiotic drive during complex speciation.” Elife 7.<br>
Meiklejohn, C. D. and Y. Tao (2010). “Genetic conflict and sex chromosome evolution.” Trends Ecol Evol 25(4): 215-223.<br>
Munch, K., K. Nam, M. H. Schierup and T. Mailund (2016). “Selective Sweeps across Twenty Millions Years of Primate Evolution.” Mol Biol Evol 33(12): 3065-3074.<br>
Nam, K., K. Munch, A. Hobolth, J. Y. Dutheil, K. R. Veeramah, A. E. Woerner, M. F. Hammer, P. Great Ape Genome Diversity, T. Mailund and M. H. Schierup (2015). “Extreme selective sweeps independently targeted the X chromosomes of the great apes.” Proc Natl Acad Sci U S A 112(20): 6413-6418.<br>
Nielsen, R., C. Bustamante, A. G. Clark, S. Glanowski, T. B. Sackton, M. J. Hubisz, A. Fledel-Alon, D. M. Tanenbaum, D. Civello, T. J. White, J. S. J, M. D. Adams and M. Cargill (2005). “A scan for positively selected genes in the genomes of humans and chimpanzees.” PLoS Biol 3(6): e170.<br>
Rathje, C. C., E. E. P. Johnson, D. Drage, C. Patinioti, G. Silvestri, N. A. Affara, C. Ialy-Radio, J. Cocquet, B. M. Skinner and P. J. I. Ellis (2019). “Differential Sperm Motility Mediates the Sex Ratio Drive Shaping Mouse Sex Chromosome Evolution.” Curr Biol 29(21): 3692-3698 e3694.<br>
Skov, L., M. C. Macia, E. Lucotte, M. I. C. Alves, D. Castellano, T. Mailund, M. H. Schierup and K. Munch (2018). “Strong selective sweeps before 45,000BP displaced archaic admixture across the human X chromosome.” bioRxiv.<br>
Skov, L., M. C. Macià, E. Lucotte, M. I. A. Cavassim, D. Castellano, T. Mailund, M. H. Schierup and K. Munch (2018). “Extreme selective sweeps displaced archaic admixture across the human X chromosome around 50,000 years ago.” bioRxiv.<br>
Stern, A. J., P. R. Wilton and R. Nielsen (2019). “An approximate full-likelihood method for inferring selection and allele frequency trajectories from DNA sequence data.” PLoS Genet 15(9): e1008384.</p>


</section>
</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../notes/meiotic-drive.html" class="pagination-link" aria-label="Meiotic drive">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-title">Meiotic drive</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../notes/phasetype.html" class="pagination-link" aria-label="Phase-type dist">
        <span class="nav-page-text"><span class="chapter-title">Phase-type dist</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>