<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Phase-type dist – Title of your project</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../notes/prdm9.html" rel="next">
<link href="../notes/notes.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/phasetype.html"><span class="chapter-title">Phase-type dist</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">Title of your project</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Project</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Notes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/chromatin.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/funding.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/hybrid-incomp.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Hybrid incomp</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/ideas.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Ideas / notes</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/mbg-exercises.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">MBG exercises</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/meiotic-drive.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Meiotic drive</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/notes.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Funding</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/phasetype.html" class="sidebar-item-text sidebar-link active"><span class="chapter-title">Phase-type dist</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/prdm9.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">PRDM9</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/sex-antagonistic.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Chromatin</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/spermatogenesis.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Spermatogenesis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/strategy.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Research strategy</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/x-brain.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">X-brain</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../notes/xi-msci.html" class="sidebar-item-text sidebar-link"><span class="chapter-title">Xi and MSCI</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul class="collapse">
  <li><a href="#ptd-applications" id="toc-ptd-applications" class="nav-link active" data-scroll-target="#ptd-applications">PtD applications</a></li>
  <li><a href="#theory" id="toc-theory" class="nav-link" data-scroll-target="#theory">Theory</a></li>
  <li><a href="#inference" id="toc-inference" class="nav-link" data-scroll-target="#inference">Inference</a></li>
  <li><a href="#im-model-admixture" id="toc-im-model-admixture" class="nav-link" data-scroll-target="#im-model-admixture">IM model admixture</a></li>
  <li><a href="#time-inhomogeneous" id="toc-time-inhomogeneous" class="nav-link" data-scroll-target="#time-inhomogeneous">Time-inhomogeneous</a></li>
  <li><a href="#demography" id="toc-demography" class="nav-link" data-scroll-target="#demography">Demography</a></li>
  <li><a href="#background-selection" id="toc-background-selection" class="nav-link" data-scroll-target="#background-selection">Background selection</a></li>
  <li><a href="#smc-transition-probabilities" id="toc-smc-transition-probabilities" class="nav-link" data-scroll-target="#smc-transition-probabilities">SMC’ transition probabilities</a></li>
  <li><a href="#applications-paper" id="toc-applications-paper" class="nav-link" data-scroll-target="#applications-paper">Applications paper</a></li>
  <li><a href="#cranconda-package" id="toc-cranconda-package" class="nav-link" data-scroll-target="#cranconda-package">CRAN/conda package</a></li>
  <li><a href="#technical" id="toc-technical" class="nav-link" data-scroll-target="#technical">Technical</a></li>
  <li><a href="#nice-papers-using-phase-type-distributions" id="toc-nice-papers-using-phase-type-distributions" class="nav-link" data-scroll-target="#nice-papers-using-phase-type-distributions"><strong>Nice papers using phase-type distributions</strong></a></li>
  <li><a href="#inhomogeneous-phase-type-distributions-and-heavy-tails" id="toc-inhomogeneous-phase-type-distributions-and-heavy-tails" class="nav-link" data-scroll-target="#inhomogeneous-phase-type-distributions-and-heavy-tails"><strong>Inhomogeneous phase-type distributions and heavy tails</strong></a></li>
  <li><a href="#fractional-coalescent" id="toc-fractional-coalescent" class="nav-link" data-scroll-target="#fractional-coalescent"><strong>Fractional coalescent</strong></a></li>
  <li><a href="#matrix-mittagleffler-distributions-and-modeling-heavy-tailed-risks" id="toc-matrix-mittagleffler-distributions-and-modeling-heavy-tailed-risks" class="nav-link" data-scroll-target="#matrix-mittagleffler-distributions-and-modeling-heavy-tailed-risks"><strong>Matrix Mittag–Leffler distributions and modeling heavy-tailed risks</strong></a></li>
  <li><a href="#the-coalescent-of-a-sample-from-a-binary-branching-process" id="toc-the-coalescent-of-a-sample-from-a-binary-branching-process" class="nav-link" data-scroll-target="#the-coalescent-of-a-sample-from-a-binary-branching-process"><strong>The coalescent of a sample from a binary branching process</strong></a></li>
  <li><a href="#full-likelihood-inference-from-the-site-frequency-spectrum-based-on-the-optimal-tree-resolution" id="toc-full-likelihood-inference-from-the-site-frequency-spectrum-based-on-the-optimal-tree-resolution" class="nav-link" data-scroll-target="#full-likelihood-inference-from-the-site-frequency-spectrum-based-on-the-optimal-tree-resolution"><strong>Full likelihood inference from the site frequency spectrum based on the optimal tree resolution</strong></a></li>
  <li><a href="#exact-calculation-of-the-joint-allele-frequency-spectrum-for-isolation-with-migration-models" id="toc-exact-calculation-of-the-joint-allele-frequency-spectrum-for-isolation-with-migration-models" class="nav-link" data-scroll-target="#exact-calculation-of-the-joint-allele-frequency-spectrum-for-isolation-with-migration-models"><strong>Exact Calculation of the Joint Allele Frequency Spectrum for Isolation with Migration Models</strong></a></li>
  <li><a href="#reduction-of-acyclic-phase-type-representations" id="toc-reduction-of-acyclic-phase-type-representations" class="nav-link" data-scroll-target="#reduction-of-acyclic-phase-type-representations"><strong>Reduction of Acyclic Phase-Type Representations</strong></a></li>
  <li><a href="#about-using-sfs-for-inference-of-admixture" id="toc-about-using-sfs-for-inference-of-admixture" class="nav-link" data-scroll-target="#about-using-sfs-for-inference-of-admixture"><strong>About using SFS for inference of admixture:</strong></a></li>
  <li><a href="#ideas-using-phase-type-distributions" id="toc-ideas-using-phase-type-distributions" class="nav-link" data-scroll-target="#ideas-using-phase-type-distributions"><strong>Ideas using phase-type distributions</strong></a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../notes/chromatin.html">Notes</a></li><li class="breadcrumb-item"><a href="../notes/phasetype.html"><span class="chapter-title">Phase-type dist</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-title">Phase-type dist</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>Find bug in RCpp version<br>
Write documentation/proof<br>
Make plans for two projects:<br>
1. inference framework<br>
2. Writing a manuscript</p>
<p><a href="https://cs.au.dk/~schwiegelshohn/">https://cs.au.dk/~schwiegelshohn/</a></p>
<p><a href="https://scholar.google.com/citations?user=X9Hl0LcAAAAJ&amp;hl=en">https://scholar.google.com/citations?user=X9Hl0LcAAAAJ&amp;hl=en</a></p>
<p>Add tests to Rcpp project<br>
Add cpp version of add_epoque</p>
<p>Write Python interface<br>
https://docs.python.org/3/extending/index.html</p>
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgoAAACCCAYAAAAnveDJAAAdp0lEQVR4Xu2dB3cUR7qG9x/dc++m4+td29drm2XBZJOjMSxgE21jG7DJiJzBNjbZBAEm5xyUkYRBIARiJSGQkJAIIgkl1623Wj3qKU1P0vSoZ+Z9zqkzM52qusJXj0Y93X8QhBBCCCE2/EFfQAghhBBiQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUIIIYTYQlEghBBCiC0UBUJcSOPrWlFf81Sl3xsa9NWEEBI1KAqEuJRL770pskcM1hcTQkhUoSgQ4lLSunYQV78Yry8mhJCoQlEgxKVQFAghboCiQIhLSetCUSCEtD8UBUJcRuGKJSJ39HCR2vk9kTmgpyja+LO+SUS5NnmiyJs6WaXCVcv01SJ3zAi17m7yDn1V2BSuXKqOmT2c12AQ4nYoCoS4jNLk7eLcG/+lROHykH6iZNtWfZOI8tuEz1R+SKV7kvXV4vzf/6jWlR8/qq8Km9JdO9Qx0zq/r68ihLgMigIhLoOiQAhxExQFQlwGRYEQ4iYoCjHCg1MnxOPcXK9E4oPqy5myPXNExbkz6jV/9vSoicKj7GyR/fFAjygULEwSNfn5al3lhbPi8ZVcjygUb9ognhUWakcInafXr4ubC+Ya59jxHfHs9i1RfvSwp183PH+u70IIaUcoCjHC9RnTPMEcCQGcxAfVGWmqTSsvnpeT8p9E9ieDoyYKJdu2iKzBvT39KmtQH/Hk2lW1rnDNCrXMFIWMXh+KZ3fuaEcIHfTdzEEfeUTh6qSxomDRfCOPvt31zQkh7QxFIUagKMQvVlG4+M5fXCsKlwf3dVwUcO6EEHdBUYgRKArxC0WBokCIm6EoxAi+RKGpvl7fjMQYTXV1FAWKAiGuhqIQAxRv3aiCtFUULg/tL15XV+ubkhijtuKByB3zidGmUgouvP3nVqKQM2KIKDtyUN+1zeBXDDkjh7a7KGT06S6yhxsXVKb3+lDcWrJA34UQ0o5QFGKAe/v2yEA6yEsUEFwpCrEPRAEyYE7EvkQha1BvUZWRpu/aZvBri6zBfdpdFDLl+UFYzH59fcZUfRdCSDtCUYgBKArxC0WBokCI26EoxAAUhfiFokBRIMTtUBSiSOWZ0+L+gX0qvbxfqq+2haIQn9Q/eSLKDu0XmQN6uUYUMvv1FMVbN4nahxWtRAEPqCrdnSxKk3eoPlyVekk/ZFBQFCJH0fp1npjy4ORxfTUhEYGiEEWqszI8AbnxxQt9tS0UhfgFk2TmQGPSdIMo4BuFjD5dReOrV61EQW0rywq5weeam8YdHEOFohA5cFGz2XYPL13QVxMSESgKUYSiQHQoChSFtkBRINGAohBFKApEh6JAUWgLFAUSDSgKUQJiEI4oNL54HheiwAf9+AbXJ7SHKDTW1tqLQu8utqKQ2b9nVESh7sljfbeEpf7pE32RB4oCiQYUhShR+7BSBfyWgNxb38QnT/Kuqcki1kWhtrJCPfDo0rtvqLsPJhKYeHHe176c4LX8t4ljRXrPzhEVhfTuHVVeZkrt9K6+iaLu8WOR3quzb1H46EPVVuZFllZRwA2RLg81fqXhlChk9OkmKs+d0XdLKNK6fOAZK1WpF/XVHigKJBpQFKIERMF6d8VQRAHbxoMomGXHJJRImHdevDl/ttfyvGlfRVwUsocN8Oon6T076ZsolCh81MVWFPDenLx1UTC3d0oUkE+iiwK+uTHbhKJA2huKQpSgKFAUKAoGFIXAUBSIm6AoRAmKAkWBomBAUQgMRYG4iYQThfzZ08X1777RFzvO7eWL1A1rWgJyYFF4ef+eyJ87M2KikD/rW3Fj5jTP5zs/rhEv7wW+8VP+3Bmyzqboi0MiUUWh+JfNqq1w3pgM7x/Y61kXKVHAhH175VKVMJFb+4mTonBz/hzx4m6JfuiAUBT8U7ThZ5HWtYOnTXRRuJkkhbOpSRRtWi/Se3TybAdReHQ5S9xetUwUb9notY8viuQ22LZa7hMsd3ftVPtUJHD7uJ1byxap2GKHiherlor//PS9vsqWhBMFXCiFv/CijfXqdiMgBxYFXJmOvywiJQq/TfxMBSATTEK4uj0QKEPumBH64pBIVFEoWv+j56/DnJFDRPnRQ551kRIFTPot/aKbVz9xUhTQp8P5NQtFwT95U78U6d3/5WkTXRQQCyAK6D8Zsg3N7SAKL4qK1Pt7+1uE1I57+39V22KfYMGj0LHP45xsfRVxCQULk8T5N/9bX+zBjBfFWzbpq2yhKEQJigJFgaJgQFHwD0WBtAWKQgSgKFAUoglFoTUUBf9QFEhboChEAIpCB/F7Q4Ooe/RIBZlX5WXqsy9+b6iX21WrySGyovBHlb9dvvGElyiMGCxKd+8UL+6Vihclxeq+CkGJglz/4NQJVWe+iCVRqK+pkZPNBc89GnyJAuoDk1ht5QPRVFenHyJuwRjB00SvThrnUxQ847avHPtVD5VQWEWh7OhhdRMtvHdaFCrPnxW1VVX6atLOIK5AIK2igDGKsQRBBxSFIKAoGN8o5I7+RKT3+Je4Mm6UtpU3OaOGqaAVaVHI/Wykvklcon+jgJsv1RQUiPJjR1S7BiUKcjt/7R1LooA7kuICvEDfKKBvWr/9SgTwyyijDXr7FAWQM+pj1W5Xxo9u9Y0Cllelpaj3TotCpmyn2ocP9dWknSmXsoixpH+jgLiSP2eGek9RCAKKghF8r339eVCicO3rSRSFNqCLAibZRBcF9a1BAFHAsXEuiUQwonB18gRbUUA9Rk8UulEUXIidKFz8vzcoCqFAUaAoRBOKgjcUBXsoCqStUBQiBEWBohBNKAreUBTs8ScKja9fq20oCsQfFIUIEUuigJstYTJ3WhTy50wXV7+coP7/qWOKAvLDDZrCBRdomWWnKFAUKAqtsROF7I8HqpuvAb+iIPuRWY+5clw/TLG/oyPW5Y4dqbYNVhSe/JYrrkz41MiLouAain7+QcVvPKzNnygglpRs2eiJF4j3D04e99rODopClAhHFHLH/tsI5g6LAu7opwa/LKOOKQqZ/XuJog3r9NVBg8nALDtFgaJAUWiNnSigPoIVBazDe/Slp9fztBxawDpzTAYrCi/vl3puQ09RcA+lyduNsdT5H35FAXHl/oF9nniB/lLt52fXVigKUYKiQFGgKFAU/EFRIOFAUXAAigJFIZpQFLyhKNhDUSDhQFFwAAQfDLzqzHT1Gf93q0q9pG0VWapSU1TAtooCgjFujlJx7rRnO9yMBsuQKs+fkZNE34iJAo6J/3WmdnpX3cHNThTwgJjq9BTPfk6JAibA6szgOmms8ig7Wz2EyyoKeDAYJKF0146EE4XGly/F45ycoO6jgGOndesoqvz8nz3eCEUUsof2V7IQjijgpk5lhw96xmTF6ZNqWSAoCu7EFIWUf74j/zBZJ+eKwUoUavLz1ZgDFIUQQSBF4C7a8JP6XJ2VIfKmfKFtFVnypn4h0uSkbBUFDDS8lh3a77UtBjrKWLx1k8iSwTJSolCybYsKOBCFq1+MtxWFkh3bvG4z7JQoZA7srf7ajmcgZPi1iP6NAuoddZxoogBwl0l1zkGJwj/Fta8m6YeIW0IRBdQPlocjCk11dSKzL77VMdoRbWr+qsIfFAV3YhUFcyxBFHI/bfmlGkUhRNwuCpgYsocPpCjEAXaikNmvR8KKAv7tR1HwTXuJAvonRSF2oSg4AEWBohAtKAqtoSjYQ1Eg4UBRcAC3igIaD2DwByUKvbvaThy+CEoU5ARGUYgcdqKAydAVotCrc9tEQZ6XKQr1NU+1HHxDUbDHEVFobNRyoSjEGxQFB3CrKFScPSVSOrytyofgjkb1JwrqiY6jh2s52WMVBRUkZNJFAZ9zZL6mKJRs/8XYjqIQFnaigIkyZ8SQdhUFNWHL9m6TKEixTOnwlprQq9JTPcfGfmndO1pya4GiYE8gUSjauF49OTJYUciSqcnH0zexjKLgHnLHj1a/WAiXQKJwZfwYcfGdv9qKAuad/Fnfakf1hqLgIlEwAmU3mboaATyAKGCbYLGKAgK9Cvo+RAF5maJQdviAJ2hRFELHnyigL7SnKGQP7RcRUTD3s4rCtckTZT/zHfgoCvYEEoXSPcmqfYIVBUhFE0XB9eTPmyUuvPVnfXHQBBSFCf5FAe9vL1+sHdUbigJFgaLgEBSF1lAU7KEoJCYUBRdCUaAoRAuKQmsoCvZQFBITioILsYoCOn7B4vnqRkQVp07om0aEilPH1fHtRCF/1neiZPtWcTNptvocKVEoO3JIBRbcTAVQFKJPJEUB2xVv3qhnofAnCjgGgoOOU6JwX4rv5WEDIiYKOHf045ob1/VDxR1uFgX0MeSfObCXsQ9FISKUnzyurjU7//c/qfoNh1BE4cbMb0XFmZOe/lKwMEm9pyhoWEWh/skT2eG7G5Pj8aP6phGh/NhhdXw7UcAFRxWnT3gGbaREofL8WbXdo+zL6jNFIfpEVBRke+eMGqZnofAnCmkfvu/zr3KnRKFkxy+qzJEUBSyH1Mc7bhYFoPqs2fYUhYiAP+hwYfOFt/4U9m3tQxEF9J+qjFRPf8Gv5/CeoqBBUaAoRAuKQmsoCvZQFBIPioJLoShQFKIFRaE1FAV7KAqJB0XBpVAUvEUhd9THPkXhVXlZVESh7olxo6l4AzciSjRRqH/2LGqi0Fhb6/U5HrATBdQPRSE+sYpCzr99j+9AUBQcgKLgLQpZMmBfHtrfOGazKGACwEWQ6FROigI6buW5M/qmcQGexpch6zGRRAF9DWV3WhSwLcZuvGErCrJ/YGKnKMQfVlFAe9xaulDfJCChikJ6c4ygKPiBotBaFGCy6pjNooB1EAWUyUlRQEeOZ1HAZJhoooCyOi0K2cMGJJ4oyLqmKMQfuiiE8+/YkEWhWQ4oCn6gKFAUogFFgaIQKhSFxIOi4FIoChSFaEBRoCiECkUh8aAouBSrKBSuWiaDZSeKAkUh4iSaKFz/booo3bvbMVG4+vk4UbBkgUrYn6LQvqKQ0bOTuLV0kb4JCQHEZvRrp0QBfwQXb1xPUQgHqyjkygkSQZGiQFGINIkmCpfluvITxx0TBdSFmRcmOYpCO4uC7CPYn4RPza0Co+87JApZsr2uT59CUQgHigJFIRpQFCgKoUJRSCwoCi6GokBRiAYUBYpCqFAUEguKgotRoiAHXeHq5TKg9aUoUBQcgaJAUQgVJ0Th2e1bakKy0kRRcAUUBRdjfqOQIQNeZv8eFAWKgiNQFCgKoeKEKJQfOyKKf9nslU8TRcEVUBRcjCkKKhhRFCgKDkFRoCiECkUhsaAouBiKAkUhGlAUKAqhQlFILCgKLiatawcvUTCDkS4Kr+oavD43Nf3u9dmkrqFJNDQ1qfeNcpsnz1sG3Ov6Ri9RSO/e0TOo02W+KigEEAWU104U8DtZ8Lss2vNXLRctPX1RJyrOGaLwMCtL1MsymqKQ8sHfvUQBZTJFIbXzP3yKAoK+VRRwzr5AHaEOfGGVpPN/+x9Z7x2VKDw4e0bVUyCa5Ema9RwI6/Fq61reNzR6769/DgYc2zxHvFr7hbVeIArqHIMQBbNf6KKQ0uFtL1FAfwXI01oXzyqrPHWri0KqDB52opDW5YPgREG2l1rnRxQyZNmtopDywVt6lgqIQlqXDh5RSHn/b7aikPqvdz2igLow80K/jwdR0PtPTdmD5jYwxpznfCEAPTqFLAoX+/byiAJigNnfm6Qo4H7/ZsxJ6/+RT1FAXLGSgT9cehgCgz6CvmZi7ft1DY1e5/Wy1juWBoP38Vreo87s4gXKG86YtsaIYLGWAXnaxkStEl++bqkLqyiomCjjQCBRePritWfso45RDlMULr33ppcoIK5QFMKgTgaXygvnRMGi+bJilojCNStUQiXNWL5X9JixX4xdc1oMWXhEDJh3SIxfe0aloYuOqmX95rYsQ/p48THRd84hMUyu/3TVKfV++JLjanukQfOPiJVzfhQFC5OMvFYulcH0mLoxDT5DTvAQkBlTvxe3li8yyrZwnri5KEmVSe2zdqW6yczNBXPUeiTsd3vlMrH9pz1iyPxDKp/RK0+oV5UWHBYrZ32vth05e7foL8/lp8Ub5THmieItG9X5I5nnf2P2d2pb/BIE5bkljz19yhq1DvsUbVqvzmHhmv2qXrp9t1edL1LXb/eKfy8/oepi4PzDMq/Dannv2QdVHaHORs7bK/K2/qLKjXzu7tyupANtMGrObjEg6ZD4ZOkx0WfOQbVvn+Z9P5p1wJMPjj1YtsHwJce82sBXQhnRXihDT9mmI5YdV3WAuhm6EG1zzLNe39cuoWzIG6/DFhllwPGGyONheV95nr1lebEMdTH665/Frc0bVTuiDo22TVJtiXrF6+0VS1T7ol/c279X3PlxtSjZsU3kz50pyo8e9rSV2Re2/LhHjJXtPHiBme9xld/UWZtF2dmzKg8zLzMVyzIsWZasytRz5n55zgfEpOVHxOFV642+L8twc8Fcz/Y3k2arMt2TfTRv2mRVjrIjB43yy+0K16xU25efOCrfLxelv+5S7Zq3YYOY+c0qcef71Wr9sV3HxMcLDqn2HyfHFNLEZYfFgVUbjH41f7ZRPjmJob+b/btg8Xxx54c1Rl+V/S5/3ixPXZh9H/tgorwyfnTMpHnj5njqAamXbAvVjguOqrZBO06btVE8OH9Wtv8MUbRxfctYl23060+7xA8zVxv9Zo0RE1BPBbLuzO0Kv1+lLtJG26ANMrbuFMtnrhVzVx9Q7YB80GeGzUgWv23couoc+13ftlPGkcOqXOjfGHfjZP/+ZOlxWc4D6nVk0j6Ru2W7KJRlQV8tlDFi57rdsj/KeCfHEvbDmMDYwFhAXua4QUxAPMT77tP3ecaLObaRzLGOGIoY3Ge2MZawz0D5OlLGGOyHYw1IOqzeY/x+2Bx/8BnLkXrO2Od1bD0Z4/aIygep39yDMmYa8UDFH1VmI4agjfS4jzGE80C9jF19Wpb5oMoX567nhfiD/MzP/eWxUBcj5L6TvlnXPBbmeGLiguW7VR2gf6AO9XIPTDqi5hsj30Mq1q2fudIzlhAvEMNLk3eoeL544VZxYN0OUbJ9qxpTGF+Fa1eIsqOHRP6cmWqbu7t2im/mbvfqn6hPlN3MO6FEwR+f/3hOvP/lTvGX0ZtVQkcw3yO9OXar6hjWZUan3K86Lz5jMsPAMtdj4pj4feCv1tEhdPMMhvT8MlUG5NVXdnYzXzXQZWMDTCg4l283XdL2tgfWjs6oM3NrqprU/zbuF09eH0xONjqwPHcMIGvd4BWBCa9l1c/1wynGrDwlBW2f2h/1gG0RPPFqrUvzvMzj+ktdZPDAK4739oRtah8EEyxDADKP8d4XLe0dKGFwo4w9ZF2a7Y2E4IJAhqDS4atdapmqC1nWZ5ZveSLB5dsPLPkagRTvEdTs/qIBe1NuqzJ1nvqresU+2fJYkeT5q3pV3yaFZU9U+fQxk3Gz3LJXePze0OD5KzpW0trP53jVAyYKvKJPm22C+rL7Ri6vuEp8vf6CvtgvJZU1amLD2O8pxxImFuQxQE42WGeCv1LNcmHSQ5/Ge4wTjEGzn6FNrdwoqfbs98FXyV77mGMMfRMxEu8Ri/759S51vnq/wHZ4NeNFh8nG8bDdG59tUccbNN8QARwfx0CMemfidlU+rENdmvtYjx0ofTjt1+a8jfhjjRd/HbNZ/aFh3R7bvSXjyqDmODVgHuYB37HJGiuRzLpAzICA6Gw4fk3VBf4YM/uFvr+Zj4pHMnZ+46dfbD51XZzOvasvbsXGE3le+aB8nabs8XymKDRDUWiBotA6URT8Q1HwnygKFAUkikKMQ1FogaLQOlEU/ENR8J8oChQFJIpCjENRaIGi0DpRFPwDUZiwtqWvo2+ibZEfXs20P+2OZa/wgCjgQj5cDBsrafW46V79yRQFjM1RK4y+6U8ULlwrFUv2GL9gChbIwKQfzrYSBbSDLgrmcsSzbtP3q/cYf0joa+jjvkTBmKSPBBQFc2J0QhRwXIxrikJr1h66IvanBh5zFIUgoSi0QFFonSgK/vE1wR3J/I/YffGWvrjthDFW2pvDGf/x6k+mKOAC3Z3nC9R7f6IQDubV/LoonM4tsf01EGIcYgYYu9qQhS/WnVOf9V+CWX/NAAHAOfgShX5zjAlx8k/nHREFyNC0TRcpCm2AohAkFIUWKAqtE0UhdBwThRikPUTBxJco2OFPFPzR0Y8oYLwAigJFIeahKLRAUWidKAqhQ1FogaJAUUCiKMQ47SkKGLyRFgV0ZhCuKOBcdCgKFIVQoCi0ECuigDiCcQZMUUBZA4EJG+fgTxRwHIw5ioIxEZv1bIWi4HLaSxRe1DaIyqcv5Wu9viog/kThfvUzNXmEIwq4MK3yCcrk/T9JigJFIRTulD8Rt8se64sTklgQBdzl717VM5XQd8sePVf9DLHAX/+qeVnn2W+UHGe+REHdQfDhM3Ep7766SZAeSxNNFLBtZkF5qxhLUXA57SUKbcFOFIyJw7hYKRxRsIOiQFEg4RELoqCDixwxhkLhq5/P+xQFE/xyAn1Qj6WJJgqIGfqvSABFweVQFAJDUaAokPCgKBhQFCgKMQ1FITAUBYoCCQ+KggFFgaIQ01AUAkNRCF4Upm9JEcUVT/XTbRNtFQX8EsasB4pCdAlGFKZsuOg6UZi7PV1f7Jd1R6+qeENRMFIsi8K4NWfE4t2XVaIoNENRCAxFIXhRyCms0E+1zbRVFK4XV4lLeffUPhSF6BKMKGCMRF4TwheFMH6IpUAeFAUjxbIo4M6bJhSFZigKgaEoUBRIeAQrCk4QriiEC0WhJVEU4gyKQmAoChQFEh4UBQOKAkUhpnFKFEavOKlnFTHsRKFfO4gCBg9FwZ2i8PPRq6ouKQrtRyBR+OuYLbZjpK3MT87yiAL6jdOigPHltCjgPCgKzonC4AUUBZ84IQr95OTtxMVJJnaigIvoGpqMiSNaopAsgx06l7Vu8EpRiBzhigL4ct15ikI74k8UcEMj8OxlZH8lYwV9EqKAOPD4+Wt9dUQpf/TcUVFA2nXhFkXBQVHYl1ro9dAvikIzTogCPreXKJhEUxSGLTIGmlk3eKUoRA6KQuziTxSiAfIxRcFpIArmJOeEKGB/ioLzomCFotAMRSEwFAWKAgkPioIBRYGiENNQFAJDUaAokPCgKBhQFCgKMQ1FITDhiALqDBdr5d9tuTDGSrREAU/Cw7J4EYW1h66ox9TifTCigIGP/kJRaB/2phSqcWAmXOSMdjiaWeT4NQNgw4lrajxhnDpNYdljsSA5U53fd5sviTUHcy3rwhMFrIcQYOzhiaSbT16nKERQFLaeueHVP/WnvlIUmqEoBCYcUUjamakfxotoiML4tWfEyn05alm8iELajbKQvlEwoSi4gzUHr6h2ePW65YIxJ6mqeaXGWrRIu3FfnR+ekGglXFH4dNUpNYlirOHJuPxGIbKiEAiKQjMUhcBQFCgKJDJQFCgKFIUYhKIQGIoCRYFEBooCRYGiEINQFAITSBTMQW7WDV4DiQIusouGKKDs/5BiEOuigCe6oZ9CFAY0B9dQReE9uT9FoX2Zuz1DtWM0RQGTbbSAKOD8fIkC+q0eSwOLwmmxYFeWGm/xIAqfLD3hUxQWynPEeYQiCpi7nIai0AxFITCBROFVXYNX3eA1kCi8lIESA8NpUQC4aCyWRcH62OrqZ7Viz6XbKr9QROHpC+MYkX4ENgmP+iDbra1EKx+TmuabR+k3kUKMOJZV1CqW+hOF/5WiYDJBjuV4EIUT2cWqLnwxZuXJoEUBFzRHA4pCMxSFwDglCvipX9REYU7sioJOOKJASHtDUTBEwY5QRAHfzkQDikIzFIXAUBQoCoS0FYpC7InC/wM/UonW5YEuigAAAABJRU5ErkJggg==" class="img-fluid"></p>
<ul>
<li></li>
<li><p>How much cheaper can we make the joint prob in the two locus arg if we assume there can be at most one recombination between to adjacent bases?<br>
</p></li>
<li><p>Just sum the joint probabilities in log space of all the ton neighbors along an alignment<br>
</p></li>
<li></li>
<li><p>Look at how distributions are generated and normalized in ptd<br>
</p></li>
<li><p>See what happens if you remove the trash states<br>
</p></li>
<li><p>I could also truncate time and still get the exact PDF up to that point</p></li>
<li><p>Very difference sizes of bottlenecks from time 1 to 2 gives quite similar time-homo distributions. So the moments must also be quite similar, especially the first ones. So maybe there is not so much information in moments when it comes to epoques unless you use a lot of them. Not so good news for Janek, I guess. I would need to fit the distributiosn not the moments for most power (or maybe both) in pymc<br>
</p></li>
<li><p>Make a 45 min lecture explaining the whole paper incl time-inhomogeneous and pop hen models<br>
</p></li>
<li><p>SMC’ with gene conversion<br>
</p></li>
<li><p><a href="https://cran.r-project.org/web/packages/maxLik/maxLik.pdf">maxLik: Maximum Likelihood Estimation and Related Tools</a><br>
</p></li>
<li><p>local B-statistic Murphey 2022<br>
</p></li>
<li><p>G12 statistic for selection Harris et al<br>
</p></li>
<li><p>Can I group all same epoque states?<br>
</p></li>
<li><p>For inference use all pairs of sites across the genome<br>
</p></li>
<li><p>Terhorst says dadi moments are samples**populations**3. So samples**6 for IM model<br>
</p></li>
<li><p>For exchangeable/unlabelled samples, the distribution of topologies is indepentent of waiting time between coalescence events. That means all information is retained in the distance between coalescence events and we can from these compute the sfs from precomputing rewards. This way we can model time-inhomogeneous coal, and two-locus args, and ghost-population. It would allow me to compute covariance matrix of a two-locus IM ghost model. With 2-locus model each state must keep track of (nr_unlinked_locus1, nr_unlinked_locus2, nr_linked). With ghost model states must keep track of how many lineages reside in which population.<br>
</p></li>
<li><p>Matrix Inversion: A fundamental operation in scientific computing | by Fernando Jean Dijkinga, M.Sc. | Medium <a href="https://medium.com/@fernando.dijkinga/matrix-inversion-a-fundamental-operation-in-scientific-computing-8217197a0e48#:~:text=The%20standard%20matrix%20inversion%20algorithm,with%20the%20number%20of%20components">Matrix Inversion: A fundamental operation in scientific computing | by Fernando Jean Dijkinga, M.Sc. | Medium</a><br>
</p></li>
<li><p><a href="https://moments.readthedocs.io/en/main/introduction.html">Introduction — moments 1.1.0 documentation</a><br>
</p></li>
<li><p>Gutenkunst Group | Gutenkunst Group <a href="https://gutengroup.mcb.arizona.edu/">https://gutengroup.mcb.arizona.edu/</a><br>
</p></li>
<li><p>Specifying a model - dadi <a href="https://dadi.readthedocs.io/en/latest/user-guide/specifying-a-model">https://dadi.readthedocs.io/en/latest/user-guide/specifying-a-model</a><br>
</p></li>
<li><p>Can I demonstrate how much information in a HMM (PSMC, TRAILS) is captured by moments and cross-moments in a two-locus model<br>
</p></li>
<li><p>Consider poisson random fields for inference<br>
</p></li>
<li><p>PtD centralized results data structure with community contributions<br>
</p></li>
<li><p>Add sparse matrix inversion to PtDalgorithms. Add general functions that takes a matrix and does inversion, should take and IPV that is (1, 1, 1, 1) by default, but can be set to (1, 0, 0, 0) to compute “first-row” inversion like tobias.<br>
</p></li>
<li><p>Maybe visit Davide and Ryan Gutenkunst<br>
</p></li>
<li><p>Bayesian model stacking<br>
</p></li>
<li><p>Lesson Explainer: Representing a System of Two Equations in Matrix Form | Nagwa: <a href="https://www.nagwa.com/en/explainers/958197090453/#:~:text=We%20recall%20that%20if%20we,the%20second%20being%20the%20%F0%9D%91%A6%20%2D">Lesson Explainer: Representing a System of Two Equations in Matrix Form | Nagwa</a><br>
</p></li>
<li><p><a href="https://cran.r-project.org/web/packages/Rcpp/vignettes/Rcpp-package.pdf">Rcpp-package.pdf https://cran.r-project.org/web/packages/Rcpp/vignettes/Rcpp-package.pdf</a><br>
</p></li>
<li><p>Faster inference of complex demographic models from large allele frequency spectra <a href="https://www.biorxiv.org/content/10.1101/2024.03.26.586844v1.full.pdf">https://www.biorxiv.org/content/10.1101/2024.03.26.586844v1.full.pdf</a><br>
</p></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2024.03.24.586479v1">A structured coalescent model reveals deep ancestral structure shared by all modern humans | bioRxiv</a><br>
</p></li>
<li><p><a href="https://math.mit.edu/~gs/linearalgebra/ila5/linearalgebra5_12-2.pdf">12.2 Covariance Matrices and Joint Probabilities</a><br>
</p></li>
<li><p>Vivaswat shastry jeremy Berg ASG likelihood<br>
</p></li>
<li><p>Ptd multi explanation: We note that, although computationally infeasible in most situations, the joint distribution function of multivariate discrete phase-type distributions can also be described by such a recursive algorithm (and time-inhomogeneous multivariate discrete phase-type distributions as seen later). Instead of defining the probability of the system visiting a vertex at time <span class="math inline">\(t\)</span>, we instead keep track of both the visited vertex and the accumulated reward at time <span class="math inline">\(t\)</span>. Consider an <span class="math inline">\(\\ell+1\)</span>-dimensional table for an <span class="math inline">\(\\ell\)</span>-dimensional multivariate discrete phase-type distribution. The first dimension has <span class="math inline">\(|V|\)</span> entries, one for each vertex. The other dimensions has entries corresponding to the natural numbers from <span class="math inline">\(0\)</span> to <span class="math inline">\(k\)</span> where <span class="math inline">\(k\)</span> is sufficiently large, such that for a given distribution function of specified accumulated rewards <span class="math inline">\(\\vec{s}\)</span>, <span class="math inline">\(F\_{\\vec{s}}\)</span>, then <span class="math inline">\(k\)</span> is larger or equal to any entry in <span class="math inline">\(\\vec{s}\)</span>. At a time <span class="math inline">\(t\)</span> an entry in the table corresponds to the probability of the system being in a specific state, and having accumulated a specific vector of integer rewards corresponding to the other dimensions. If we have accumulated more than any of the rewards <span class="math inline">\(\\vec{s}\)</span>, we assign probability zero to the entry in the table before the next time step. Assuming we have removed the vertices with zero rewards in all dimensions (e.g.&nbsp;using the reward transformation algorithm), then all time steps will increase at least one entry in the reward accumulation, and after a finite number of steps all entries will either be zero, or for the absorbing vertices the total probability of the system having reached an absorbing state with less than or equal to the rewards <span class="math inline">\(\\vec{s}\)</span>, which is the definition of the cumulative distribution function for the multivariate discrete phase-type distribution.<br>
</p></li>
<li><p>Is it possible to fit a ptd to a tree sequence from relate i.e the topology and coal levels for each SNP distinguishing ancestral and derived? That could produce popgen parameters of some complex demographic model. It could only take a few samples but would produce posteriors if made with Lyme<br>
</p></li>
<li><p>Read ASMC paper to understand how ARG-needle builds an arg from unphased SNPs and for inspiration for<br>
</p></li>
<li></li>
</ul>
<section id="ptd-applications" class="level2">
<h2 class="anchored" data-anchor-id="ptd-applications">PtD applications</h2>
<ul>
<li><a href="https://github.com/leovan/quarto-pseudocode">Quarto Pseudocode Extension</a><br>
</li>
<li>R for macOS Developers <a href="https://mac.r-project.org/">https://mac.r-project.org/</a><br>
</li>
<li>Make a test notebook to see if I can write the new version of the ptd manuscript in a quarto notebook, incl algorithms, inline math and refs<br>
</li>
<li>Davis Vaughan - Using VS Code to Debug R Packages with C or C++ Code <a href="https://blog.davisvaughan.com/posts/2022-03-11-using-vs-code-to-debug-r-packages-with-c-code/">https://blog.davisvaughan.com/posts/2022-03-11-using-vs-code-to-debug-r-packages-with-c-code/</a><br>
</li>
<li>Roxygen to Markdown • roxygen2md <a href="https://roxygen2md.r-lib.org/">https://roxygen2md.r-lib.org/</a><br>
</li>
<li>Documenting Rcpp functions and classes in R packages | R-bloggers <a href="https://www.r-bloggers.com/2020/01/documenting-rcpp-functions-and-classes-in-r-packages/">https://www.r-bloggers.com/2020/01/documenting-rcpp-functions-and-classes-in-r-packages/</a><br>
</li>
<li>PTD: Fitting covariance is fine for SFS butto do a rolling regression with one-hot-vectors I need the joint probabilities to implement a dist in pymc<br>
</li>
<li>Multidimensionel ptd: We note that, although computationally infeasible in most situations, the joint distribution function of multivariate discrete phase-type distributions can also be described by such a recursive algorithm (and time-inhomogeneous multivariate discrete phase-type distributions as seen later). Similar to the marginal distribution, each dimension of a multivariate distribution is defined by a the probability accumulated In addition to keeping track of the probability that the system visits a vertex at time <span class="math inline">\(t\)</span>, we also need to keep track of the accumulated reward at each state at that time. For an <span class="math inline">\(\\ell+1\)</span>-dimensional distribution, we can do this using an <span class="math inline">\(\\ell\)</span>-dimensional table for each state. The multidimensional index of each entry represents a multidimensional integer reward. Each table dimension <span class="math inline">\(k\)</span> is sufficiently large , such that for a given distribution function of specified accumulated rewards <span class="math inline">\(\\vec{s}\)</span>, <span class="math inline">\(F\_{\\vec{s}}\)</span>, then <span class="math inline">\(k\)</span> is larger or equal to any entry in <span class="math inline">\(\\vec{s}\)</span>. At any time <span class="math inline">\(t\)</span> each entry holds the probability of staying at that vertex.<br>
</li>
<li>Add state_space() function that just takes an initial state and callback function. The latter should take a state vector and return a dict/rlist with child states and corresponding rates.<br>
</li>
<li>How much information can I extract with unphased data (single locus) and multiple moments? How much with two-locus? How much info is stored in a five-sequence two-locus model compared to a five-sequence HMM?<br>
</li>
<li>Figure out what Gravel is doing with Asger (read weakly structured stem.. paper)<br>
</li>
<li>Read “Efficient computation of the joint sample frequency spectra for multiple populations” for PtD inspiration<br>
</li>
<li>Zoom with Mathias Steinrucken<br>
</li>
<li></li>
</ul>
</section>
<section id="theory" class="level2">
<h2 class="anchored" data-anchor-id="theory">Theory</h2>
<ul>
<li></li>
<li>Is Tobias already computing someging equivalent to van Loen with his “higher moments are expectations with other rewards” trick? And then with time inhom, just doing that multiple likes like Janek multiplying epoques? Does it show that there is a way to do van Loen on a time inhom on a single matrix like maybe I do?<br>
</li>
<li>Think about whether the accmulated SFS also represent the branch length variances, covariances, etc. How can it be estimated empirically with only one mutation per tree?<br>
</li>
<li>probability - Proving that MGF determines PDF when the PDF is defined for whole real line - Cross Validated <a href="https://stats.stackexchange.com/questions/46062/proving-that-mgf-determines-pdf-when-the-pdf-is-defined-for-whole-real-line">https://stats.stackexchange.com/questions/46062/proving-that-mgf-determines-pdf-when-the-pdf-is-defined-for-whole-real-line</a><br>
</li>
<li>Implement multidimensional rewards to get full distribution.<br>
</li>
<li></li>
</ul>
</section>
<section id="inference" class="level2">
<h2 class="anchored" data-anchor-id="inference">Inference</h2>
<ul>
<li></li>
<li>Is it possible to solve the problem that method-of-moments does not always have a solution, by not using a fixed sample moment estimates, but distributions of those estimators taking sample variance into account?<br>
</li>
<li>Just wrap a multinomial dist with one sample. (Joint sfs can be serialized to vector) to get a custom dist parameterized by popgen parameters<br>
</li>
<li>Pymc inference - Given model parameters, compute marginal ton expectations - Use expectations as parameters for a custom pymc distribution that has one-hot encoded vectors as outcomes. Maybe it would be a hypergeometric with a sample off one?<br>
</li>
<li>Method of moments estimation <a href="https://web.stanford.edu/class/archive/cs/cs109/cs109.1218/files/student_drive/7.3.pdf">https://web.stanford.edu/class/archive/cs/cs109/cs109.1218/files/student_drive/7.3.pdf</a><br>
</li>
<li>PDF, CDF and Moments of Special Distributions - LNTwww <a href="https://en.lntwww.de/Applets:PDF,_CDF_and_Moments_of_Special_Distributions">https://en.lntwww.de/Applets:PDF,_CDF_and_Moments_of_Special_Distributions</a><br>
</li>
<li>Tons should be one hot encoded and each have a discrete distribution with two outcomes: seeing the ton (phase type computed ton freq times mutation rate) and not seeing the ton (vast majority of prob). Those two numbers can be precomputed for a grid of parameters. It also means that their prior is a single sample binomial. Ideally, the custom distribution should sample one-hot vectors of presence/absence of tons. So that a one-hot vector is a single observation. See if it is possible to have vectors as single outcomes<br>
</li>
<li>Inference should be done on single SNPs as already implemented for selection. because assumption of no recombination between is not valid. Pairs of SNPs should be done with two-locus ARG<br>
</li>
<li></li>
<li>Stern Nielsen paper nr two<br>
</li>
<li></li>
</ul>
</section>
<section id="im-model-admixture" class="level2">
<h2 class="anchored" data-anchor-id="im-model-admixture">IM model admixture</h2>
<ul>
<li></li>
<li>Try the 2pop 2locus example with just two samples<br>
</li>
<li>Try turning your 2-locus island model into a time inhom IM model<br>
</li>
<li>Try to turn my Rolling regression example into a custom dist<br>
</li>
<li>IM model idea:<br>
</li>
<li>Pre-compute discrete expectation and covariance matrices for a grid of combinations of T, N1, N2, NA, M in the IM model.<br>
</li>
<li>Assume two models:<br>
</li>
<li>1. Global model with the fitted T, N1, N2, NA, M<br>
</li>
<li>2. Same but with M=0<br>
</li>
<li></li>
<li></li>
<li>Use the models to emit the distributions used by pymc.<br>
</li>
<li>The parameters to be explained should be the SNP by SNP “expectation” and “covariance” matrices (which are ofcause 0 or 1), and the explaining variables should be those controlling the mix between global M and M=0.<br>
</li>
<li>Do rolling regression to fit the mixture between the two models using the trick of Juraj.<br>
</li>
<li>Should produce a landscape showing some regions that are exclusively M=0.<br>
</li>
<li></li>
<li></li>
<li>OR<br>
</li>
<li>- Pre-compute discrete expectation and covariance matrices for a grid of combinations of T, N1, N2, NA, M in the IM model.<br>
</li>
<li>- The parameters to be explained should be the SNP by SNP “expectation” and “covariance” matrices (which are ofcause 0 or 1)<br>
</li>
<li>- The explaining variables should be a global T, global N1, global N2, global Na, local M<br>
</li>
<li></li>
<li></li>
<li>Demographically explicit scans for barriers to gene flow using gIMble <a href="https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:3df635f4-91dc-4729-86df-69cced2a933a">https://www.readcube.com/library/ff394a98-20f7-4882-871c-840e9f73d65b:3df635f4-91dc-4729-86df-69cced2a933a</a><br>
</li>
<li>Idea: more elegant version of gimble using phase-types and rolling regression,<br>
</li>
<li>Run 2D-SFS for a grid of parameters<br>
</li>
<li>Use 2D-SFS as distribution for pymc.<br>
</li>
<li>Make the three population sizes and the split time be globally estimated variables and allow only the two migration rates to vary along the genome<br>
</li>
<li></li>
<li></li>
<li>Rolling regression with phasetypes: If you do an rolling regression with a custum pymc dist, then the prior for the values to be explained cannot be a multidimentional (with a dimention for each ton, becuase we only have marginals), it would have to be a vector of explained values (tons) each with its own custum dist. But then I would need to normalize the tons by mutation rate the the ton occurences can be compared across regions. For starters, try to make a rolling regression with a simple standard SFS as function of population size, and see if the posterior predictions for each ton look sensible.<br>
</li>
<li></li>
<li>Maybe use phase types for the Shriram / Monty outgroup conditioned uniform SFS<br>
</li>
<li>Chen 2007 abstract: “We develop the theory for computing the joint frequency spectra of alleles in two closely related species. We allow for arbitrary population growth in both species after they had a common ancestor. We focus on the case in which a single chromosome is sequenced from one of the species. We use classical diffusion theory to show that, if the ancestral species was at equilibrium under mutation and drift and a chromosome from one of the descendant species carries the derived allele, the frequency spectrum in the other species is uniform, independently of the demographic history of both species. We also predict the expected densities of segregating and fixed sites when the chromosome from the other species carries the ancestral allele. We compare the predictions of our model with the site-frequency spectra of SNPs in the four HapMap populations of humans when the nucleotide present in the Neanderthal DNA sequence is ancestral or derived, using the chimp genome as the outgroup.<br>
</li>
<li></li>
<li></li>
<li>I guess it is a special case if the IM model: E.g., 10 from pop1 and 1 from pop2. The prob of a 1,1 ton then should be the same as 2,1 3,1 4,1 and so on<br>
</li>
<li></li>
<li></li>
<li>It is because the normal SFS proportions each are multiplied with relative branch levels in the ancestral pop tree where each ton would need to fall<br>
</li>
<li></li>
<li></li>
<li>See if Monty has variances in his paper. Otherwise it could be a topic for a small paper on the coalescent and SFS properties.<br>
</li>
<li></li>
<li></li>
<li>Or maybe expand the model to two locus ARG for a stronger test of ghost introgression. I guess a particular covariance structure is expected under the null<br>
</li>
<li></li>
<li></li>
<li>Maybe there are more fun special cases like that with two on pop2?<br>
</li>
<li></li>
<li></li>
<li>Read Quibl paper for insparation<br>
</li>
<li>Try to make an IM model with an admixture pulse. Then compute the ILS prop as function of allele frequency<br>
</li>
<li></li>
</ul>
</section>
<section id="time-inhomogeneous" class="level2">
<h2 class="anchored" data-anchor-id="time-inhomogeneous">Time-inhomogeneous</h2>
<ul>
<li></li>
<li>Add a 1 parameter for each epoque so i can Update parameters og all epoques latet<br>
</li>
<li>Would be good with a 0 1 state vector specifying which slots should be collapsed when mapping states to sister states. E.g the slot specifying population could be 0 specifying that the states for each population should map to a single state in the ancestral population. I could multiply that array with the identified sister state to zero out the population label. Then states from each population would map to the same sister state in the ancestral population<br>
</li>
<li>assert is not working in add_epoque! Use throw or something else…<br>
</li>
<li>Fix add_epoque bug<br>
</li>
<li>Find out how to raise Rcpp exception from c code (not cpp code). Maybe I can just use the cpp method: throw(Rcpp::exception(“My Error Message”,“bar.cpp”,4));<br>
</li>
<li>Make sure the PDF for my new approach matches the PDF obtained by updating parameters many times.<br>
</li>
<li></li>
</ul>
</section>
<section id="demography" class="level2">
<h2 class="anchored" data-anchor-id="demography">Demography</h2>
<ul>
<li>Try to implement PSMC and PSMC with structure<br>
</li>
<li></li>
<li>Positive/negative selection on sampled variant<br>
</li>
<li></li>
<li>I guess I can compute the probability of the ratio of derived and ancestral branch when det freq is non-zero. That Not for time inhom but maybe the expectation is good enough if I use pymc<br>
</li>
<li>Compare classical fixation time result to ptd computation with derived variant<br>
</li>
<li>Try to plot the SFS, PDFs, moments, etc for the coalescent with selection and compare to the standard coalescent in R<br>
</li>
<li>My selection model (using sampled relate trees) is just like clues except I don’t need to use importance sampling because I can sample from the coalescent with selection and compare to neutral model<br>
</li>
<li>Use log10(pop size)-1 binsize so it is 1 for 100 2 for 1000 3 for 10,000 and so on<br>
</li>
<li>Try a very large selection coefficient and test if I mostly get singletons then<br>
</li>
<li>Is it possible to distinguish N and s if we know the distributions of tons (maybe thomas already does): …since different combinations of N and s give different fixation time? The distribution of a ton is smaller with smaller N because the fixation time is short<br>
</li>
<li>Think about using phase-types for the selection graph - maybe two locus<br>
</li>
<li></li>
</ul>
</section>
<section id="background-selection" class="level2">
<h2 class="anchored" data-anchor-id="background-selection">Background selection</h2>
<ul>
<li></li>
<li>Outline BGS model with ptd<br>
</li>
<li>All you have to do to model background selection (assuming infinite sites model) is … add selection to one locus in a two-locus arg. Then sample only ancestral, but to allow the freq to go from N to N-1 with the mutation rate - and then model the frequency transitions like otherwise for selection. This only models two sites and not any nonlinear cummulative effect of many sites, but at least for the expectation I think the cummulative effect could be taken care of by adjusting the N<br>
</li>
<li></li>
</ul>
</section>
<section id="smc-transition-probabilities" class="level2">
<h2 class="anchored" data-anchor-id="smc-transition-probabilities">SMC’ transition probabilities</h2>
<ul>
<li></li>
<li>Phase-type idea for computing conditional probabilities of lefr and right marginal trees in an SMC’<br>
</li>
<li>1. Make a normal coalescent-only graph for left marginal (representing both left and right trees before the recombination event)<br>
</li>
<li>2. Compute (easy analytically) probability of the recombination landing on each branch at a particular time<br>
</li>
<li>3. Use those probabilities as init prob vector for a two-locus coalescent-only ARG graph starting from the recombination point (lineages must be labelled). All lineages have connected loci except the recombined one.<br>
</li>
<li></li>
<li></li>
<li></li>
</ul>
</section>
<section id="applications-paper" class="level2">
<h2 class="anchored" data-anchor-id="applications-paper">Applications paper</h2>
<ul>
<li></li>
<li>Think about using phase-types for two locus covariance<br>
</li>
<li>When doing an empirical two locus sfs, is it then ok just to know the AVERAGE recombination rate between the loci? -if variance is the MEAN squared deviation from the mean.<br>
</li>
<li>Contributions to LD from covariance of different ton branches at two loci. Connection between LD and branch covariance matrix<br>
</li>
<li>Plot matrix sparseness for time inhom matices<br>
</li>
<li></li>
</ul>
</section>
<section id="cranconda-package" class="level2">
<h2 class="anchored" data-anchor-id="cranconda-package">CRAN/conda package</h2>
<ul>
<li></li>
<li>Move ptd repo to munch group and add quarto pages for docs and examples<br>
</li>
<li>Make seperate popgen ptd repo with coal models. Add ptd algorithm repo as sub module<br>
</li>
<li>CRAN<br>
</li>
<li></li>
</ul>
</section>
<section id="technical" class="level2">
<h2 class="anchored" data-anchor-id="technical">Technical</h2>
<ul>
<li></li>
<li>Gimble<br>
</li>
<li>This might be useful for visualzing differential evolution optimization: <a href="https://notebooks.gesis.org/binder/jupyter/user/bqplot-bqplot-gallery-7bx3cse0/voila/render/notebooks/gradient_descent.ipynb?token=EoiSIUHnTGeD67DxVhaHSg">https://notebooks.gesis.org/binder/jupyter/user/bqplot-bqplot-gallery-7bx3cse0/voila/render/notebooks/gradient_descent.ipynb?token=EoiSIUHnTGeD67DxVhaHSg</a><br>
</li>
<li>Lesson Explainer: Representing a System of Two Equations in Matrix Form | Nagwa: <a href="https://www.nagwa.com/en/explainers/958197090453/#:~:text=We%20recall%20that%20if%20we,the%20second%20being%20the%20%F0%9D%91%A6%20%2D">https://www.nagwa.com/en/explainers/958197090453/#:~:text=We%20recall%20that%20if%20we,the%20second%20being%20the%20%F0%9D%91%A6%20%2D</a><br>
</li>
<li>How to make a python front for Ptdalgorithms<br>
</li>
<li>Write a script that reads RcppExports.cpp and convertes<br>
</li>
<li></li>
<li></li>
<li>SEXP create_graph(size_t state_length);<br>
</li>
<li>RcppExport SEXP _ptdalgorithms_create_graph(SEXP state_lengthSEXP) {<br>
</li>
<li></li>
<li></li>
<li>it into someting like<br>
</li>
<li></li>
<li></li>
<li>def create_graph(*args, **kwargs):<br>
</li>
<li>return ptdalgorithms.create_graph(*args, **kwargs)<br>
</li>
<li>initialize = create_graph<br>
</li>
<li></li>
<li></li>
<li>Make a setup.py like for SAP.<br>
</li>
<li></li>
<li></li>
<li>Make a conda package<br>
</li>
<li></li>
<li></li>
<li></li>
<li>Tobias old document<br>
</li>
<li></li>
<li></li>
<li></li>
<li></li>
<li>Nice papers using phase-type distributions<br>
</li>
<li></li>
<li>Inhomogeneous phase-type distributions and heavy tails<br>
</li>
<li>https://arxiv.org/abs/1812.04139<br>
</li>
<li>Defining IPH<br>
</li>
<li></li>
<li>Fractional coalescent<br>
</li>
<li><a href="https://www.pnas.org/content/116/13/6244.abstract">https://www.pnas.org/content/116/13/6244.abstract</a><br>
</li>
<li>Derives the f-coalescent and discusses when it can be useful. Not directly related to the phase-type distribution, but I am working with the article and one asked for the link, so here it is anyways.<br>
</li>
<li></li>
<li>Matrix Mittag–Leffler distributions and modeling heavy-tailed risks<br>
</li>
<li><a href="https://arxiv.org/pdf/1906.05316.pdf">https://arxiv.org/pdf/1906.05316.pdf</a><br>
</li>
<li>Defines the matrix Mittag-Leffler function, other Mittag-Leffler functions and the inhomogeneous phase-type distribution.<br>
</li>
<li></li>
<li></li>
<li>The coalescent of a sample from a binary branching process<br>
</li>
<li>Not PTDs but possibly good for inspiration<br>
</li>
<li><a href="https://reader.elsevier.com/reader/sd/pii/S0040580918300753?token=111BD111B246F3126F0D20A941A7DBE6E0368C489684F7322336C8D05FFD245E757B1EF39A42492F80CBCA804F754C71">https://reader.elsevier.com/reader/sd/pii/S0040580918300753?token=111BD111B246F3126F0D20A941A7DBE6E0368C489684F7322336C8D05FFD245E757B1EF39A42492F80CBCA804F754C71</a><br>
</li>
<li></li>
<li>Full likelihood inference from the site frequency spectrum based on the optimal tree resolution<br>
</li>
<li>Not PTDs but alternative take on full likelihood of SFS<br>
</li>
<li><a href="https://reader.elsevier.com/reader/sd/pii/S004058091730165X?token=0761F0937D456B6093EEC099BC68F3BF464D0996DBCC91BF2D718906EB2EB278E225F7A858E908D8089A17268C94909E">https://reader.elsevier.com/reader/sd/pii/S004058091730165X?token=0761F0937D456B6093EEC099BC68F3BF464D0996DBCC91BF2D718906EB2EB278E225F7A858E908D8089A17268C94909E</a><br>
</li>
<li></li>
<li>Exact Calculation of the Joint Allele Frequency Spectrum for Isolation with Migration Models<br>
</li>
<li>Not PTDs but but exact SFS for IM model (but without distribution I think)<br>
</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5586375/pdf/241.pdf">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5586375/pdf/241.pdf</a><br>
</li>
<li></li>
<li>Reduction of Acyclic Phase-Type Representations<br>
</li>
<li><a href="https://www.researchgate.net/publication/265231177_Reduction_of_Acyclic_Phase-Type_Representations">https://www.researchgate.net/publication/265231177_Reduction_of_Acyclic_Phase-Type_Representations</a><br>
</li>
<li></li>
<li></li>
<li>About using SFS for inference of admixture:<br>
</li>
<li>Sankararaman: <a href="https://advances.sciencemag.org/content/6/7/eaax5097">https://advances.sciencemag.org/content/6/7/eaax5097</a><br>
</li>
<li>Which site Slatkin: <a href="https://www.genetics.org/content/genetics/177/1/387.full.pdf">https://www.genetics.org/content/genetics/177/1/387.full.pdf</a><br>
</li>
<li>Which cite Nielsen: <a href="https://www.genetics.org/content/genetics/154/2/931.full.pdf">https://www.genetics.org/content/genetics/154/2/931.full.pdf</a><br>
</li>
<li></li>
<li></li>
<li>Ideas using phase-type distributions<br>
</li>
<li></li>
<li>Can misspecification of the coalescent process be mistaken for changes in historical population size? Simulate a dataset using a nest-site model and estimate historical population sizes assuming a Kingman coalescent. If estimated population sizes increase backwards in time, it would be valuable to such processes using Mittag-Leffler distributions.<br>
</li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>
</section>
<section id="nice-papers-using-phase-type-distributions" class="level2">
<h2 class="anchored" data-anchor-id="nice-papers-using-phase-type-distributions"><strong>Nice papers using phase-type distributions</strong></h2>
</section>
<section id="inhomogeneous-phase-type-distributions-and-heavy-tails" class="level2">
<h2 class="anchored" data-anchor-id="inhomogeneous-phase-type-distributions-and-heavy-tails"><strong>Inhomogeneous phase-type distributions and heavy tails</strong></h2>
<p><a href="https://arxiv.org/abs/1812.04139">https://arxiv.org/abs/1812.04139</a><br>
Defining IPH</p>
</section>
<section id="fractional-coalescent" class="level2">
<h2 class="anchored" data-anchor-id="fractional-coalescent"><strong>Fractional coalescent</strong></h2>
<p><a href="https://www.pnas.org/content/116/13/6244.abstract">https://www.pnas.org/content/116/13/6244.abstract</a><br>
Derives the f-coalescent and discusses when it can be useful. Not directly related to the phase-type distribution, but I am working with the article and one asked for the link, so here it is anyways.</p>
</section>
<section id="matrix-mittagleffler-distributions-and-modeling-heavy-tailed-risks" class="level2">
<h2 class="anchored" data-anchor-id="matrix-mittagleffler-distributions-and-modeling-heavy-tailed-risks"><strong>Matrix Mittag–Leffler distributions and modeling heavy-tailed risks</strong></h2>
<p><a href="https://arxiv.org/pdf/1906.05316.pdf">https://arxiv.org/pdf/1906.05316.pdf</a><br>
Defines the matrix Mittag-Leffler function, other Mittag-Leffler functions and the inhomogeneous phase-type distribution.</p>
</section>
<section id="the-coalescent-of-a-sample-from-a-binary-branching-process" class="level2">
<h2 class="anchored" data-anchor-id="the-coalescent-of-a-sample-from-a-binary-branching-process"><strong>The coalescent of a sample from a binary branching process</strong></h2>
<p>Not PTDs but possibly good for inspiration<br>
<a href="https://reader.elsevier.com/reader/sd/pii/S0040580918300753?token=111BD111B246F3126F0D20A941A7DBE6E0368C489684F7322336C8D05FFD245E757B1EF39A42492F80CBCA804F754C71">https://reader.elsevier.com/reader/sd/pii/S0040580918300753?token=111BD111B246F3126F0D20A941A7DBE6E0368C489684F7322336C8D05FFD245E757B1EF39A42492F80CBCA804F754C71</a></p>
</section>
<section id="full-likelihood-inference-from-the-site-frequency-spectrum-based-on-the-optimal-tree-resolution" class="level2">
<h2 class="anchored" data-anchor-id="full-likelihood-inference-from-the-site-frequency-spectrum-based-on-the-optimal-tree-resolution"><strong>Full likelihood inference from the site frequency spectrum based on the optimal tree resolution</strong></h2>
<p>Not PTDs but alternative take on full likelihood of SFS<br>
<a href="https://reader.elsevier.com/reader/sd/pii/S004058091730165X?token=0761F0937D456B6093EEC099BC68F3BF464D0996DBCC91BF2D718906EB2EB278E225F7A858E908D8089A17268C94909E">https://reader.elsevier.com/reader/sd/pii/S004058091730165X?token=0761F0937D456B6093EEC099BC68F3BF464D0996DBCC91BF2D718906EB2EB278E225F7A858E908D8089A17268C94909E</a></p>
</section>
<section id="exact-calculation-of-the-joint-allele-frequency-spectrum-for-isolation-with-migration-models" class="level2">
<h2 class="anchored" data-anchor-id="exact-calculation-of-the-joint-allele-frequency-spectrum-for-isolation-with-migration-models"><strong>Exact Calculation of the Joint Allele Frequency Spectrum for Isolation with Migration Models</strong></h2>
<p>Not PTDs but but exact SFS for IM model (but without distribution I think)<br>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5586375/pdf/241.pdf">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5586375/pdf/241.pdf</a></p>
</section>
<section id="reduction-of-acyclic-phase-type-representations" class="level2">
<h2 class="anchored" data-anchor-id="reduction-of-acyclic-phase-type-representations"><strong>Reduction of Acyclic Phase-Type Representations</strong></h2>
<p><a href="https://www.researchgate.net/publication/265231177_Reduction_of_Acyclic_Phase-Type_Representations">https://www.researchgate.net/publication/265231177_Reduction_of_Acyclic_Phase-Type_Representations</a></p>
</section>
<section id="about-using-sfs-for-inference-of-admixture" class="level2">
<h2 class="anchored" data-anchor-id="about-using-sfs-for-inference-of-admixture"><strong>About using SFS for inference of admixture:</strong></h2>
<p>Sankararaman: <a href="https://advances.sciencemag.org/content/6/7/eaax5097">https://advances.sciencemag.org/content/6/7/eaax5097</a><br>
Which site Slatkin: <a href="https://www.genetics.org/content/genetics/177/1/387.full.pdf">https://www.genetics.org/content/genetics/177/1/387.full.pdf</a><br>
Which cite Nielsen: <a href="https://www.genetics.org/content/genetics/154/2/931.full.pdf">https://www.genetics.org/content/genetics/154/2/931.full.pdf</a></p>
</section>
<section id="ideas-using-phase-type-distributions" class="level2">
<h2 class="anchored" data-anchor-id="ideas-using-phase-type-distributions"><strong>Ideas using phase-type distributions</strong></h2>
<ul>
<li>Can misspecification of the coalescent process be mistaken for changes in historical population size? Simulate a dataset using a nest-site model and estimate historical population sizes assuming a Kingman coalescent. If estimated population sizes increase backwards in time, it would be valuable to such processes using Mittag-Leffler distributions.<br>
</li>
<li></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../notes/notes.html" class="pagination-link" aria-label="Funding">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-title">Funding</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../notes/prdm9.html" class="pagination-link" aria-label="PRDM9">
        <span class="nav-page-text"><span class="chapter-title">PRDM9</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>